<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0053)https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/ -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2614560/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Multiresolution Bilateral Filtering for Image Denoising</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="./Multiresolution Bilateral Filtering for Image Denoising_files/jig.min.css"><script type="text/javascript" src="./Multiresolution Bilateral Filtering for Image Denoising_files/jig.min.js.download"></script><script data-dapp-detection="">
(function() {
  let alreadyInsertedMetaTag = false

  function __insertDappDetected() {
    if (!alreadyInsertedMetaTag) {
      const meta = document.createElement('meta')
      meta.name = 'dapp-detected'
      document.head.appendChild(meta)
      alreadyInsertedMetaTag = true
    }
  }

  if (window.hasOwnProperty('web3')) {
    // Note a closure can't be used for this var because some sites like
    // www.wnyc.org do a second script execution via eval for some reason.
    window.__disableDappDetectionInsertion = true
    // Likely oldWeb3 is undefined and it has a property only because
    // we defined it. Some sites like wnyc.org are evaling all scripts
    // that exist again, so this is protection against multiple calls.
    if (window.web3 === undefined) {
      return
    }
    if (!window.web3.currentProvider ||
        !window.web3.currentProvider.isMetaMask) {
      __insertDappDetected()
    }
  } else {
    var oldWeb3 = window.web3
    Object.defineProperty(window, 'web3', {
      configurable: true,
      set: function (val) {
        if (!window.__disableDappDetectionInsertion)
          __insertDappDetected()
        oldWeb3 = val
      },
      get: function () {
        if (!window.__disableDappDetectionInsertion)
          __insertDappDetected()
        return oldWeb3
      }
    })
  }
})()</script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="IEEE transactions on image processing : a publication of the IEEE Signal Processing Society"><meta name="citation_title" content="Multiresolution Bilateral Filtering for Image Denoising"><meta name="citation_authors" content="Ming Zhang, Bahadir K. Gunturk"><meta name="citation_date" content="December 2008"><meta name="citation_issue" content="12"><meta name="citation_volume" content="17"><meta name="citation_firstpage" content="2324"><meta name="citation_doi" content="10.1109/TIP.2008.2006658"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2614560/?report=abstract"><meta name="citation_pmid" content="19004705"><meta name="DC.Title" content="Multiresolution Bilateral Filtering for Image Denoising"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Ming Zhang"><meta name="DC.Contributor" content="Bahadir K. Gunturk"><meta name="DC.Date" content="2008 Dec"><meta name="DC.Identifier" content="10.1109/TIP.2008.2006658"><meta name="DC.Language" content="en"><meta property="og:title" content="Multiresolution Bilateral Filtering for Image Denoising"><meta property="og:type" content="article"><meta property="og:description" content="The bilateral filter is a nonlinear filter that does spatial averaging without smoothing edges; it has shown to be an effective image denoising technique. An important issue with the application of the bilateral filter is the selection of the filter parameters, ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="./Multiresolution Bilateral Filtering for Image Denoising_files/pmc.min.css" type="text/css"><link rel="stylesheet" href="./Multiresolution Bilateral Filtering for Image Denoising_files/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="./Multiresolution Bilateral Filtering for Image Denoising_files/common.min.js.download">//</script><script type="text/javascript" src="./Multiresolution Bilateral Filtering for Image Denoising_files/NcbiTagServer.min.js.download">//</script><meta name="citationexporter" content="backend:&#39;https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/&#39;"><script type="text/javascript" src="./Multiresolution Bilateral Filtering for Image Denoising_files/jquery.citationexporter.min.js.download">//</script><link rel="stylesheet" href="./Multiresolution Bilateral Filtering for Image Denoising_files/citationexporter.css" type="text/css"><script type="text/javascript" src="./Multiresolution Bilateral Filtering for Image Denoising_files/MathJax.js.download"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/epub/"><link rel="alternate" type="application/pdf" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/pdf/nihms83523.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="https://www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE898CB2DF507F310000000005EA036D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="./Multiresolution Bilateral Filtering for Image Denoising_files/4143404.css"><link type="text/css" rel="stylesheet" href="./Multiresolution Bilateral Filtering for Image Denoising_files/4157116.css" media="print"><script async="1" src="./Multiresolution Bilateral Filtering for Image Denoising_files/analytics.js.download"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main-bold; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main-italic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Math-italic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style><style type="text/css">@font-face {font-family: MathJax_Script; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Script-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Script-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="https://www.ncbi.nlm.nih.gov/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="./Multiresolution Bilateral Filtering for Image Denoising_files/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="https://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="https://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="./Multiresolution Bilateral Filtering for Image Denoising_files/emyncbi.html" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="https://www.ncbi.nlm.nih.gov/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="https://www.ncbi.nlm.nih.gov/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:&#39;pmc-search-autocomplete&#39;,disableUrl:&#39;NcbiSearchBarAutoComplCtrl&#39;" autocomplete="off" data-sbconfig="ds:&#39;no&#39;,pjs:&#39;no&#39;,afs:&#39;yes&#39;" aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" style="visibility: hidden;"><img src="./Multiresolution Bilateral Filtering for Image Denoising_files/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="https://www.ncbi.nlm.nih.gov/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="https://www.ncbi.nlm.nih.gov/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC2614560</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="./Multiresolution Bilateral Filtering for Image Denoising_files/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden&#39;" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" role="menuitem" aria-expanded="false" aria-haspopup="true">IEEE Trans Image Process</a></span>. Author manuscript; available in PMC 2009 Dec 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="https://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19004705" target="pmc_ext" ref="reftype=publisher&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">IEEE Trans Image Process. 2008 Dec; 17(12): 2324–2333. </span></a></div> <span class="doi">doi:&nbsp;<a href="https://dx.doi.org/10.1109%2FTIP.2008.2006658" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1109/TIP.2008.2006658</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2614560</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS83523</span></div><div class="fm-citation-pmid">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19004705">19004705</a></div></div></div></div><h1 class="content-title">Multiresolution Bilateral Filtering for Image Denoising</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Zhang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19004705">Ming Zhang</a> and  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Gunturk%20BK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19004705">Bahadir K. Gunturk</a></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" class="pmctoggle" rid="idm139639980505232_ai">Author information</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" class="pmctoggle" rid="idm139639980505232_cpl">Copyright and License information</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139639980505232_ai" style="display:none"><div class="fm-affl" lang="en" id="A1">Department of Electrical and Computer Engineering, Louisiana State University, Baton Rouge, LA 70803, Emails: <a href="mailto:dev@null" data-email="ude.usl@4gnahzm" class="oemail">ude.usl@4gnahzm</a>, <a href="mailto:dev@null" data-email="ude.usl.ece@ridahab" class="oemail">ude.usl.ece@ridahab</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139639980505232_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="https://www.ncbi.nlm.nih.gov/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="https://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19004705" target="pmc_ext" ref="reftype=publisher&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">IEEE Trans Image Process</a></div><div>See other articles in PMC that <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm139640070984272" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139640070984272title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">The bilateral filter is a nonlinear filter that does spatial averaging without smoothing edges; it has shown to be an effective image denoising technique. An important issue with the application of the bilateral filter is the selection of the filter parameters, which affect the results significantly. There are two main contributions of this paper. The first contribution is an empirical study of the optimal bilateral filter parameter selection in image denoising applications. The second contribution is an extension of the bilateral filter: multiresolution bilateral filter, where bilateral filtering is applied to the approximation (low-frequency) subbands of a signal decomposed using a wavelet filter bank. The multiresolution bilateral filter is combined with wavelet thresholding to form a new image denoising framework, which turns out to be very effective in eliminating noise in real noisy images. Experimental results with both simulated and real data are provided.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">I. Introduction</h2><p id="P2" class="p p-first">There are different sources of noise in a digital image. Some noise components, such as the dark signal nonuniformity (DSNU) and the photoresponse nonuniformity (PRNU), display nonuniform spatial characteristics. This type of noise is often referred as fixed pattern noise (FPN) because the underlying spatial pattern is not time varying. Temporal noise, on the other hand, does not have a fixed spatial pattern. Dark current and photon shot noise, read noise, and reset noise are examples of temporal noise. The overall noise characteristics in an image depend on many factors, including sensor type, pixel dimensions, temperature, exposure time, and ISO speed. Noise is in general space varying and channel dependent. Blue channel is typically the noisiest channel due to the low transmittance of blue filters. In single-chip digital cameras, demosaicking algorithms are used to interpolate missing color components; hence, noise is not necessarily uncorrelated for different pixels. An often neglected characteristic of image noise is the spatial frequency. Referring to <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>, noise may have low-frequency (coarse-grain) and high-frequency (fine-grain) fluctuations. High-frequency noise is relatively easier to remove; on the other hand, it is difficult to distinguish between real signal and low-frequency noise.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139640002624304" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f1.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f1.jpg"></a></div><div id="largeobj_idm139640002624304" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p1">Portion of an image captured with a Sony DCR-TRV27, and its red, green, and blue channels are shown in raster scan order. The blue channel is the most degraded channel; it has a coarse-grain noise characteristics. The red and green channels have finer-grain noise characteristics.</p></div></div></div><p id="P3">Many denoising methods have been developed over the years; among these methods, wavelet thresholding is one of the most popular approaches. In wavelet thresholding, a signal is decomposed into its approximation (low-frequency) and detail (high-frequency) subbands; since most of the image information is concentrated in a few large coefficients, the detail subbands are processed with hard or soft thresholding operations. The critical task in wavelet thresholding is the threshold selection. Various threshold selection strategies have been proposed, for example, VisuShrink [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R1" rid="R1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">1</a>], SureShrink [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R2" rid="R2" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">2</a>], and BayesShrink [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502065" role="button" aria-expanded="false" aria-haspopup="true">3</a>]. In the VisuShrink approach, a universal threshold that is a function of the noise variance and the number of samples is developed based on the minimax error measure. The threshold value in the SureShrink approach is optimal in terms of the Stein’s unbiased risk estimator. The BayesShrink approach determines the threshold value in a Bayesian framework, through modeling the distribution of the wavelet coefficients as Gaussian. These shrinkage methods have later been improved by considering inter-scale and intra-scale correlations of the wavelet coefficients [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502081" role="button" aria-expanded="false" aria-haspopup="true">4</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502077" role="button" aria-expanded="false" aria-haspopup="true">5</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R6" rid="R6" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">6</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R7" rid="R7" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">7</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502070" role="button" aria-expanded="false" aria-haspopup="true">8</a>]. The method in [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502075" role="button" aria-expanded="false" aria-haspopup="true">4</a>] models the neighborhoods of coefficients at adjacent positions and scales as Gaussian scale mixture and applies the Bayesian least squares estimation technique to update the wavelet coefficients. The method, known as the BLS-GSM method, is one of the benchmarks in the denoising literature due to its outstanding PSNR performance. Some recent methods have surpassed the PSNR performance of [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502066" role="button" aria-expanded="false" aria-haspopup="true">4</a>]. Among these methods, [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R9" rid="R9" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">9</a>] constructs a global field of Gaussian scale mixtures to model subbands of wavelet coefficients as a product of two independent homogeneous Gaussian Markov random fields, and develops an iterative denoising algorithm. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R10" rid="R10" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">10</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502085" role="button" aria-expanded="false" aria-haspopup="true">11</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502100" role="button" aria-expanded="false" aria-haspopup="true">12</a>] develop gray-scale and color image denoising algorithms based on sparse and redundant representations over learned dictionaries, where training and denoising use the K-SVD algorithm. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R13" rid="R13" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">13</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502072" role="button" aria-expanded="false" aria-haspopup="true">14</a>] group 2D image fragments into 3D data arrays, and apply a collaborative filtering procedure, which consists of 3D transformation, shrinkage of the transform spectrum, and inverse 3D transformation. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502071" role="button" aria-expanded="false" aria-haspopup="true">15</a>] models an ideal image patch as a linear combination of noisy image patches and formulates a total least squares estimation algorithm.</p><p id="P4">A recently popular denosing method is the bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>]. While the term “bilateral filter” was coined in [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>], variants of it have been published as the sigma filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R17" rid="R17" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">17</a>], the neighborhood filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R18" rid="R18" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">18</a>], and the SUSAN filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R19" rid="R19" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">19</a>]. The bilateral filter takes a weighted sum of the pixels in a local neighborhood; the weights depend on both the spatial distance and the intensity distance. In this way, edges are preserved well while noise is averaged out. Mathematically, at a pixel location <strong>x</strong>, the output of the bilateral filter is calculated as follows: 
</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="M1" overflow="scroll" role="math" style="width: 21.373em; display: inline-block;"><span style="display: inline-block; position: relative; width: 17.24em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(0.305em, 1017.19em, 3.833em, -999.997em); top: -2.266em; left: 0.003em;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="mover" id="MathJax-Span-4"><span style="display: inline-block; position: relative; width: 0.96em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.51em, 4.136em, -999.997em); top: -3.979em; left: 0.154em;"><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math-italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.48em, 1000.71em, 3.984em, -999.997em); top: -4.635em; left: 0.204em;"><span class="mo" id="MathJax-Span-6" style="font-family: MathJax_Main;">∼</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-7" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-8" style="font-family: MathJax_Main-bold;">x</span><span class="mo" id="MathJax-Span-9" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-10" style="font-family: MathJax_Main; padding-left: 0.305em;">=</span><span class="mfrac" id="MathJax-Span-11" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 0.859em; height: 0px; margin-right: 0.103em; margin-left: 0.103em;"><span style="position: absolute; clip: rect(3.178em, 1000.41em, 4.136em, -999.997em); top: -4.635em; left: 50%; margin-left: -0.249em;"><span class="mn" id="MathJax-Span-12" style="font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.274em; left: 50%; margin-left: -0.401em;"><span class="mi" id="MathJax-Span-13" style="font-family: MathJax_Math-italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(0.859em, 1000.86em, 1.212em, -999.997em); top: -1.258em; left: 0.003em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.003em; border-top: 1.3px solid; width: 0.859em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span><span class="munder" id="MathJax-Span-14" style="padding-left: 0.154em;"><span style="display: inline-block; position: relative; width: 2.724em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1001.41em, 4.589em, -999.997em); top: -3.979em; left: 0.658em;"><span class="mo" id="MathJax-Span-15" style="font-family: MathJax_Size2; vertical-align: 0.003em;">∑</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.279em, 1002.67em, 4.388em, -999.997em); top: -2.82em; left: 0.003em;"><span class="mrow" id="MathJax-Span-16"><span class="mi" id="MathJax-Span-17" style="font-size: 70.7%; font-family: MathJax_Main-bold;">y</span><span class="mo" id="MathJax-Span-18" style="font-size: 70.7%; font-family: MathJax_Main;">∈</span><span class="mi" id="MathJax-Span-19" style="font-size: 70.7%; font-family: MathJax_Script;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.204em;"></span></span><span class="mo" id="MathJax-Span-20" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-21" style="font-size: 70.7%; font-family: MathJax_Main-bold;">x</span><span class="mo" id="MathJax-Span-22" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mrow" id="MathJax-Span-23" style="padding-left: 0.154em;"><span class="msup" id="MathJax-Span-24"><span style="display: inline-block; position: relative; width: 3.077em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-25"><span class="mi" id="MathJax-Span-26" style="font-family: MathJax_Math-italic;">e</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.836em; left: 0.456em;"><span class="mrow" id="MathJax-Span-27"><span class="mfrac" id="MathJax-Span-28"><span style="display: inline-block; position: relative; width: 2.271em; height: 0px; margin-right: 0.103em; margin-left: 0.103em;"><span style="position: absolute; clip: rect(3.329em, 1002.17em, 4.236em, -999.997em); top: -4.433em; left: 50%; margin-left: -1.106em;"><span class="mrow" id="MathJax-Span-29"><span class="mo" id="MathJax-Span-30" style="font-size: 50%; font-family: MathJax_Main;">−</span><span class="msup" id="MathJax-Span-31"><span style="display: inline-block; position: relative; width: 1.767em; height: 0px;"><span style="position: absolute; clip: rect(3.48em, 1001.41em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-32"><span class="mrow" id="MathJax-Span-33"><span class="mrow" id="MathJax-Span-34"><span class="mo" id="MathJax-Span-35" style="vertical-align: 0.003em;"><span><span style="font-size: 50%; font-family: MathJax_Main;">∥</span></span></span><span class="mrow" id="MathJax-Span-36"><span class="mi" id="MathJax-Span-37" style="font-size: 50%; font-family: MathJax_Main-bold;">y</span><span class="mo" id="MathJax-Span-38" style="font-size: 50%; font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-39" style="font-size: 50%; font-family: MathJax_Main-bold;">x</span></span><span class="mo" id="MathJax-Span-40" style="vertical-align: 0.003em;"><span><span style="font-size: 50%; font-family: MathJax_Main;">∥</span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.181em; left: 1.515em;"><span class="mn" id="MathJax-Span-41" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.329em, 1000.86em, 4.388em, -999.997em); top: -3.475em; left: 50%; margin-left: -0.451em;"><span class="mrow" id="MathJax-Span-42"><span class="mn" id="MathJax-Span-43" style="font-size: 50%; font-family: MathJax_Main;">2</span><span class="msubsup" id="MathJax-Span-44"><span style="display: inline-block; position: relative; width: 0.607em; height: 0px;"><span style="position: absolute; clip: rect(3.632em, 1000.3em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-45"><span class="mi" id="MathJax-Span-46" style="font-size: 50%; font-family: MathJax_Math-italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.48em, 1000.3em, 4.136em, -999.997em); top: -4.131em; left: 0.355em;"><span class="mn" id="MathJax-Span-47" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.48em, 1000.3em, 4.136em, -999.997em); top: -3.727em; left: 0.305em;"><span class="mi" id="MathJax-Span-48" style="font-size: 50%; font-family: MathJax_Math-italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(0.859em, 1002.27em, 1.212em, -999.997em); top: -1.207em; left: 0.003em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.003em; border-top: 1.3px solid; width: 2.271em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msup" id="MathJax-Span-49"><span style="display: inline-block; position: relative; width: 4.136em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-50"><span class="mi" id="MathJax-Span-51" style="font-family: MathJax_Math-italic;">e</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.735em; left: 0.456em;"><span class="mrow" id="MathJax-Span-52"><span class="mfrac" id="MathJax-Span-53"><span style="display: inline-block; position: relative; width: 3.38em; height: 0px; margin-right: 0.103em; margin-left: 0.103em;"><span style="position: absolute; clip: rect(3.329em, 1003.23em, 4.236em, -999.997em); top: -4.433em; left: 50%; margin-left: -1.61em;"><span class="mrow" id="MathJax-Span-54"><span class="mo" id="MathJax-Span-55" style="font-size: 50%; font-family: MathJax_Main;">−</span><span class="mo" id="MathJax-Span-56" style="font-size: 50%; font-family: MathJax_Main;">∣</span><span class="mi" id="MathJax-Span-57" style="font-size: 50%; font-family: MathJax_Math-italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span class="mo" id="MathJax-Span-58" style="font-size: 50%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-59" style="font-size: 50%; font-family: MathJax_Main-bold;">y</span><span class="mo" id="MathJax-Span-60" style="font-size: 50%; font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-61" style="font-size: 50%; font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-62" style="font-size: 50%; font-family: MathJax_Math-italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span class="mo" id="MathJax-Span-63" style="font-size: 50%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-64" style="font-size: 50%; font-family: MathJax_Main-bold;">x</span><span class="mo" id="MathJax-Span-65" style="font-size: 50%; font-family: MathJax_Main;">)</span><span class="msup" id="MathJax-Span-66" style=""><span style="display: inline-block; position: relative; width: 0.406em; height: 0px;"><span style="position: absolute; clip: rect(3.48em, 1000.1em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-67"><span class="mo" id="MathJax-Span-68" style="vertical-align: 0.003em;"><span><span style="font-size: 50%; font-family: MathJax_Main;">∣</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.181em; left: 0.154em;"><span class="mn" id="MathJax-Span-69" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.329em, 1000.86em, 4.287em, -999.997em); top: -3.475em; left: 50%; margin-left: -0.451em;"><span class="mrow" id="MathJax-Span-70"><span class="mn" id="MathJax-Span-71" style="font-size: 50%; font-family: MathJax_Main;">2</span><span class="msubsup" id="MathJax-Span-72"><span style="display: inline-block; position: relative; width: 0.607em; height: 0px;"><span style="position: absolute; clip: rect(3.632em, 1000.3em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-73"><span class="mi" id="MathJax-Span-74" style="font-size: 50%; font-family: MathJax_Math-italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.48em, 1000.3em, 4.136em, -999.997em); top: -4.131em; left: 0.355em;"><span class="mn" id="MathJax-Span-75" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.632em, 1000.25em, 4.136em, -999.997em); top: -3.828em; left: 0.305em;"><span class="mi" id="MathJax-Span-76" style="font-size: 50%; font-family: MathJax_Math-italic;">r</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(0.859em, 1003.38em, 1.212em, -999.997em); top: -1.207em; left: 0.003em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.003em; border-top: 1.3px solid; width: 3.38em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mi" id="MathJax-Span-77" style="font-family: MathJax_Math-italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span class="mo" id="MathJax-Span-78" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-79" style="font-family: MathJax_Main-bold;">y</span><span class="mo" id="MathJax-Span-80" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-81" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.271em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.809em; border-left: 0px solid; width: 0px; height: 4.191em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><mrow><mover accent="true"><mi>I</mi><mo>∼</mo></mover><mo stretchy="false">(</mo><mi mathvariant="bold">x</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mn>1</mn><mi>C</mi></mfrac><munder><mo>∑</mo><mrow><mi mathvariant="bold">y</mi><mo>∈</mo><mi mathvariant="script">N</mi><mo stretchy="false">(</mo><mi mathvariant="bold">x</mi><mo stretchy="false">)</mo></mrow></munder><mrow><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><msup><mrow><mrow><mrow><mo>‖</mo><mrow><mi mathvariant="bold">y</mi><mo>−</mo><mi mathvariant="bold">x</mi></mrow><mo>‖</mo></mrow></mrow></mrow><mn>2</mn></msup></mrow><mrow><mn>2</mn><msubsup><mrow><mi>σ</mi></mrow><mi>d</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></msup><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><mo>∣</mo><mi>I</mi><mo stretchy="false">(</mo><mi mathvariant="bold">y</mi><mo stretchy="false">)</mo><mo>−</mo><mi>I</mi><mo stretchy="false">(</mo><mi mathvariant="bold">x</mi><mo stretchy="false">)</mo><msup><mrow><mo>∣</mo></mrow><mn>2</mn></msup></mrow><mrow><mn>2</mn><msubsup><mrow><mi>σ</mi></mrow><mi>r</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></msup><mi>I</mi><mo stretchy="false">(</mo><mi mathvariant="bold">y</mi><mo stretchy="false">)</mo></mrow><mo>,</mo></mrow></math></script></div><div class="l">(1)</div></div><p> where <em>σ<sub>d</sub></em> and <em>σ<sub>r</sub></em> are parameters controlling the fall-off of the weights in spatial and intensity domains, respectively, 
<img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523ig1.jpg" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523ig1.jpg"> (<strong>x</strong>) is a spatial neighborhood of <strong>x</strong>, and <em>C</em> is the normalization constant: 
</p><div class="disp-formula" id="FD2"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style=""><nobr><span class="math" id="M2" overflow="scroll" role="math" style="width: 15.577em; display: inline-block;"><span style="display: inline-block; position: relative; width: 12.553em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(0.204em, 1012.45em, 3.732em, -999.997em); top: -2.165em; left: 0.003em;"><span class="mrow" id="MathJax-Span-83"><span class="mrow" id="MathJax-Span-84"><span class="mi" id="MathJax-Span-85" style="font-family: MathJax_Math-italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span class="mo" id="MathJax-Span-86" style="font-family: MathJax_Main; padding-left: 0.305em;">=</span><span class="munder" id="MathJax-Span-87" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 2.724em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1001.41em, 4.589em, -999.997em); top: -3.979em; left: 0.658em;"><span class="mo" id="MathJax-Span-88" style="font-family: MathJax_Size2; vertical-align: 0.003em;">∑</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.279em, 1002.67em, 4.388em, -999.997em); top: -2.82em; left: 0.003em;"><span class="mrow" id="MathJax-Span-89"><span class="mi" id="MathJax-Span-90" style="font-size: 70.7%; font-family: MathJax_Main-bold;">y</span><span class="mo" id="MathJax-Span-91" style="font-size: 70.7%; font-family: MathJax_Main;">∈</span><span class="mi" id="MathJax-Span-92" style="font-size: 70.7%; font-family: MathJax_Script;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.204em;"></span></span><span class="mo" id="MathJax-Span-93" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-94" style="font-size: 70.7%; font-family: MathJax_Main-bold;">x</span><span class="mo" id="MathJax-Span-95" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mrow" id="MathJax-Span-96" style="padding-left: 0.154em;"><span class="msup" id="MathJax-Span-97"><span style="display: inline-block; position: relative; width: 3.077em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-98"><span class="mi" id="MathJax-Span-99" style="font-family: MathJax_Math-italic;">e</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.836em; left: 0.456em;"><span class="mrow" id="MathJax-Span-100"><span class="mfrac" id="MathJax-Span-101"><span style="display: inline-block; position: relative; width: 2.271em; height: 0px; margin-right: 0.103em; margin-left: 0.103em;"><span style="position: absolute; clip: rect(3.329em, 1002.17em, 4.236em, -999.997em); top: -4.433em; left: 50%; margin-left: -1.106em;"><span class="mrow" id="MathJax-Span-102"><span class="mo" id="MathJax-Span-103" style="font-size: 50%; font-family: MathJax_Main;">−</span><span class="msup" id="MathJax-Span-104"><span style="display: inline-block; position: relative; width: 1.767em; height: 0px;"><span style="position: absolute; clip: rect(3.48em, 1001.41em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-105"><span class="mrow" id="MathJax-Span-106"><span class="mrow" id="MathJax-Span-107"><span class="mo" id="MathJax-Span-108" style="vertical-align: 0.003em;"><span><span style="font-size: 50%; font-family: MathJax_Main;">∥</span></span></span><span class="mrow" id="MathJax-Span-109"><span class="mi" id="MathJax-Span-110" style="font-size: 50%; font-family: MathJax_Main-bold;">y</span><span class="mo" id="MathJax-Span-111" style="font-size: 50%; font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-112" style="font-size: 50%; font-family: MathJax_Main-bold;">x</span></span><span class="mo" id="MathJax-Span-113" style="vertical-align: 0.003em;"><span><span style="font-size: 50%; font-family: MathJax_Main;">∥</span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.181em; left: 1.515em;"><span class="mn" id="MathJax-Span-114" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.329em, 1000.86em, 4.388em, -999.997em); top: -3.475em; left: 50%; margin-left: -0.451em;"><span class="mrow" id="MathJax-Span-115"><span class="mn" id="MathJax-Span-116" style="font-size: 50%; font-family: MathJax_Main;">2</span><span class="msubsup" id="MathJax-Span-117"><span style="display: inline-block; position: relative; width: 0.607em; height: 0px;"><span style="position: absolute; clip: rect(3.632em, 1000.3em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-118"><span class="mi" id="MathJax-Span-119" style="font-size: 50%; font-family: MathJax_Math-italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.48em, 1000.3em, 4.136em, -999.997em); top: -4.131em; left: 0.355em;"><span class="mn" id="MathJax-Span-120" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.48em, 1000.3em, 4.136em, -999.997em); top: -3.727em; left: 0.305em;"><span class="mi" id="MathJax-Span-121" style="font-size: 50%; font-family: MathJax_Math-italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(0.859em, 1002.27em, 1.212em, -999.997em); top: -1.207em; left: 0.003em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.003em; border-top: 1.3px solid; width: 2.271em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msup" id="MathJax-Span-122"><span style="display: inline-block; position: relative; width: 4.136em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-123"><span class="mi" id="MathJax-Span-124" style="font-family: MathJax_Math-italic;">e</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.735em; left: 0.456em;"><span class="mrow" id="MathJax-Span-125"><span class="mfrac" id="MathJax-Span-126"><span style="display: inline-block; position: relative; width: 3.38em; height: 0px; margin-right: 0.103em; margin-left: 0.103em;"><span style="position: absolute; clip: rect(3.329em, 1003.23em, 4.236em, -999.997em); top: -4.433em; left: 50%; margin-left: -1.61em;"><span class="mrow" id="MathJax-Span-127"><span class="mo" id="MathJax-Span-128" style="font-size: 50%; font-family: MathJax_Main;">−</span><span class="mo" id="MathJax-Span-129" style="font-size: 50%; font-family: MathJax_Main;">∣</span><span class="mi" id="MathJax-Span-130" style="font-size: 50%; font-family: MathJax_Math-italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span class="mo" id="MathJax-Span-131" style="font-size: 50%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-132" style="font-size: 50%; font-family: MathJax_Main-bold;">y</span><span class="mo" id="MathJax-Span-133" style="font-size: 50%; font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-134" style="font-size: 50%; font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-135" style="font-size: 50%; font-family: MathJax_Math-italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span class="mo" id="MathJax-Span-136" style="font-size: 50%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-137" style="font-size: 50%; font-family: MathJax_Main-bold;">x</span><span class="mo" id="MathJax-Span-138" style="font-size: 50%; font-family: MathJax_Main;">)</span><span class="msup" id="MathJax-Span-139" style=""><span style="display: inline-block; position: relative; width: 0.406em; height: 0px;"><span style="position: absolute; clip: rect(3.48em, 1000.1em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-140"><span class="mo" id="MathJax-Span-141" style="vertical-align: 0.003em;"><span><span style="font-size: 50%; font-family: MathJax_Main;">∣</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.181em; left: 0.154em;"><span class="mn" id="MathJax-Span-142" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.329em, 1000.86em, 4.287em, -999.997em); top: -3.475em; left: 50%; margin-left: -0.451em;"><span class="mrow" id="MathJax-Span-143"><span class="mn" id="MathJax-Span-144" style="font-size: 50%; font-family: MathJax_Main;">2</span><span class="msubsup" id="MathJax-Span-145"><span style="display: inline-block; position: relative; width: 0.607em; height: 0px;"><span style="position: absolute; clip: rect(3.632em, 1000.3em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-146"><span class="mi" id="MathJax-Span-147" style="font-size: 50%; font-family: MathJax_Math-italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.48em, 1000.3em, 4.136em, -999.997em); top: -4.131em; left: 0.355em;"><span class="mn" id="MathJax-Span-148" style="font-size: 50%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.632em, 1000.25em, 4.136em, -999.997em); top: -3.828em; left: 0.305em;"><span class="mi" id="MathJax-Span-149" style="font-size: 50%; font-family: MathJax_Math-italic;">r</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(0.859em, 1003.38em, 1.212em, -999.997em); top: -1.207em; left: 0.003em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.003em; border-top: 1.3px solid; width: 3.38em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-150" style="font-family: MathJax_Main;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.17em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.809em; border-left: 0px solid; width: 0px; height: 4.191em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><mrow><mi>C</mi><mo>=</mo><munder><mo>∑</mo><mrow><mi mathvariant="bold">y</mi><mo>∈</mo><mi mathvariant="script">N</mi><mo stretchy="false">(</mo><mi mathvariant="bold">x</mi><mo stretchy="false">)</mo></mrow></munder><mrow><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><msup><mrow><mrow><mrow><mo>‖</mo><mrow><mi mathvariant="bold">y</mi><mo>−</mo><mi mathvariant="bold">x</mi></mrow><mo>‖</mo></mrow></mrow></mrow><mn>2</mn></msup></mrow><mrow><mn>2</mn><msubsup><mrow><mi>σ</mi></mrow><mi>d</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></msup><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><mo>∣</mo><mi>I</mi><mo stretchy="false">(</mo><mi mathvariant="bold">y</mi><mo stretchy="false">)</mo><mo>−</mo><mi>I</mi><mo stretchy="false">(</mo><mi mathvariant="bold">x</mi><mo stretchy="false">)</mo><msup><mrow><mo>∣</mo></mrow><mn>2</mn></msup></mrow><mrow><mn>2</mn><msubsup><mrow><mi>σ</mi></mrow><mi>r</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></msup></mrow><mo>.</mo></mrow></math></script></div><div class="l">(2)</div></div><p></p><p id="P5">Although the bilateral filter was first proposed as an intuitive tool, recent papers have pointed out the connections with some well established techniques. In [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502059" role="button" aria-expanded="false" aria-haspopup="true">20</a>], it is shown that the bilateral filter is identical to the first iteration of the Jacobi algorithm (diagonal normalized steepest descent) with a specific cost function. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R21" rid="R21" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">21</a>] and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R22" rid="R22" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">22</a>] relate the bilateral filter with the anisotropic diffusion. The bilateral filter can also be viewed as an Euclidean approximation of the Beltrami flow, which produces a spectrum of image enhancement algorithms ranging from the <em>L</em><sub>2</sub> linear diffusion to the <em>L</em><sub>1</sub> nonlinear flows [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502064" role="button" aria-expanded="false" aria-haspopup="true">23</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R24" rid="R24" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">24</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502102" role="button" aria-expanded="false" aria-haspopup="true">25</a>]. In [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R22" rid="R22" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">22</a>], Buades et al. proposes a nonlocal means filter, where similarity of local patches is used in determining the pixel weights. When the patch size is reduced to one pixel, the nonlocal means filter becomes equivalent to the bilateral filter. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502079" role="button" aria-expanded="false" aria-haspopup="true">26</a>] extends the work of [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R22" rid="R22" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">22</a>] by controlling the neighborhood of each pixel adaptively.</p><p id="P6">In addition to image denoising, the bilateral filter has also been used in some other applications, including tone mapping [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R27" rid="R27" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">27</a>], image enhancement [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R28" rid="R28" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">28</a>], volumetric denoising [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R29" rid="R29" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">29</a>], exposure correction [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R30" rid="R30" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">30</a>], shape and detail enhancement from multiple images [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R31" rid="R31" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">31</a>], and retinex [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R32" rid="R32" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">32</a>]. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R27" rid="R27" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">27</a>] describes a fast implementation of the bilateral filter; the implementation is based on a piecewise-linear approximation in the intensity domain and appropriate sub-sampling in the spatial domain. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R33" rid="R33" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">33</a>] later derives an improved acceleration scheme for the filter through expressing it in a higher-dimensional space where the signal intensity is added as the third dimension to the spatial domain.</p><p id="P7" class="p p-last">Although the bilateral filter is being used more and more widely, there is not much theoretical basis on selecting the optimal <em>σ<sub>d</sub></em> and <em>σ<sub>r</sub></em> values. These parameters are typically selected by trial and error. In Section II, we empirically analyze these parameters as a function of noise variance for image denoising applications. We will show that the value of <em>σ<sub>r</sub></em> is more critical than the value of <em>σ<sub>d</sub></em>; we will in fact show that the optimal value of <em>σ<sub>r</sub></em> (in the mean square error sense) is linearly proportional to the standard deviation of the noise. In Section III, we will propose an extension of the bilateral filter. We will argue that the image denoising performance of the bilateral filter can be improved by incorporating it into a multiresolution framework. This will be demonstrated in Section IV with simulations and real data experiments.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">II. Parameter selection for the bilateral filter</h2><p id="P8" class="p p-first">There are two parameters that control the behavior of the bilateral filter. Referring to (1), <em>σ<sub>d</sub></em> and <em>σ<sub>r</sub></em> characterizes the spatial and intensity domain behaviors, respectively. In case of image denoising applications, the question of selecting optimal parameter values has not been answered completely from a theoretical perspective. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R22" rid="R22" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">22</a>] analyzes the behavior of the bilateral filter depending on the derivative of the input signal and <em>σ<sub>r</sub></em>/<em>σ<sub>d</sub></em> values. Conditions under which the filter behaves like a Gaussian filter, anisotropic filter, and shock filter were examined. [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502090" role="button" aria-expanded="false" aria-haspopup="true">26</a>] proposes an adaptive neighborhood size selection method for the non-local means algorithm, which can be considered as a generalization of the bilateral filter. The neighborhood size is chosen to minimize the upper bound of the local <em>L</em><sub>2</sub> risk; however, the effect of the intensity domain parameter is not considered. In this section, we provide an empirical study of optimal parameter values as a function of noise variance, and we will see that the intensity domain parameter <em>σ<sub>r</sub></em> is more critical than the spatial domain parameter <em>σ<sub>d</sub></em>.</p><p id="P9">To understand the relationship among <em>σ<sub>d</sub></em>, <em>σ<sub>r</sub></em>, and the noise standard deviation <em>σ<sub>n</sub></em>, the following experiments were done. Zero-mean white Gaussian noise was added to some test images and the bilateral filter was applied for different values of the parameters <em>σ<sub>d</sub></em> and <em>σ<sub>r</sub></em>. The experiment was repeated for different noise variances and the mean squared error (MSE) values were recorded. The average MSE values are given in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>. Examining these plots, it can be seen that the optimal <em>σ<sub>d</sub></em> value is relatively insensitive to noise variance compared to the optimal <em>σ<sub>r</sub></em> value. It appears that a good range for the <em>σ<sub>d</sub></em> value is roughly [1.5 – 2.1]; on the other hand, the optimal <em>σ<sub>r</sub></em> value changes significantly as the noise standard deviation <em>σ<sub>n</sub></em> changes. This is an expected result because if <em>σ<sub>r</sub></em> is smaller than <em>σ<sub>n</sub></em>, noisy data could remain isolated and untouched as in the case of the salt-and-pepper noise problem of the bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>]. When <em>σ<sub>r</sub></em> is sufficiently large, <em>σ<sub>d</sub></em> becomes important; apparently, increasing the value of <em>σ<sub>d</sub></em> too much results in over-smoothing and decrease of MSE.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139639985265728" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f2.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f2.jpg"></a></div><div id="largeobj_idm139639985265728" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p2">The MSE values between the original image and the denoised image for different values of <em>σ<sub>d</sub></em>, <em>σ<sub>r</sub></em>, and the noise standard deviation <em>σ<sub>n</sub></em> are displayed. The results displayed are average results for 200 images. The number of samples along the <em>σ<sub>d</sub></em> (and <em>σ<sub>r</sub></em>) direction is 10; the results are interpolated to produce smoother plots.</p></div></div></div><p id="P10" class="p p-last">To see the relationship between <em>σ<sub>n</sub></em> and the optimal <em>σ<sub>r</sub></em>, we set <em>σ<sub>d</sub></em> to some constant values, and determined the optimal <em>σ<sub>r</sub></em> values (minimizing MSE) as a function of <em>σ<sub>n</sub></em>. The experiments were again repeated for a set of images; the average values and the standard deviations are displayed in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>. We can make several observations from these plots: (1) Optimal <em>σ<sub>r</sub></em> and <em>σ<sub>n</sub></em> are linearly related to a large degree. (2) The standard deviation from the mean increases for larger values of <em>σ<sub>n</sub></em>. (3) When <em>σ<sub>d</sub></em> value is increased, the linearity between the optimal <em>σ<sub>r</sub></em> and <em>σ<sub>n</sub></em> still holds, but with lower slope. Obviously, there is no single value for (<em>σ<sub>r</sub></em>/<em>σ<sub>n</sub></em>) that is optimal for all images and <em>σ<sub>d</sub></em> values; in fact, future research should look for spatially adaptive parameter selection to take local texture characteristics into account. On the other hand, these experiments at least tell us some guidelines in selecting these parameters.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139640037684176" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f3.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f3.jpg"></a></div><div id="largeobj_idm139640037684176" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="__p3">The optimal <em>σ<sub>r</sub></em> values as a function of the noise standard deviation <em>σ<sub>n</sub></em> are plotted based on the experiments with 200 test images. The blue data points are the mean of optimal <em>σ<sub>r</sub></em> values that produce the smallest MSE for each <em>σ<sub>n</sub></em> value. The blue vertical lines denote the standard deviation of the optimal <em>σ<sub>r</sub></em> for the test images. The least squares fits to the means of the optimal <em>σ<sub>r</sub></em>/<em>σ<sub>n</sub></em> data are plotted as red lines. The slopes of these lines are, from left to right, 2.56, 2.16, and 1.97.</p></div></div></div></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title">III. A multiresolution image denoising framework</h2><p id="P11" class="p p-first">As we have discussed in Section I, image noise is not necessarily white and may have different spatial frequency (fine-grain and coarse-grain) characteristics. Multiresolution analysis has been proven to be an important tool for eliminating noise in signals; it is possible to distinguish between noise and image information better at one resolution level than another. Images in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a> motivate the use of the bilateral filter in a multiresolution framework; in that figure, approximation subbands of a real noisy image are displayed. It is seen that the coarse-grain noise becomes fine-grain as the image is decomposed further into its subbands. While it is not possible to get rid of the coarse-grain noise at the highest level, it could be eliminated at a lower level.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139640000018400" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f4.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f4.jpg"></a></div><div id="largeobj_idm139640000018400" class="largeobj-link align_right" style=""><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a7--><div class="caption"><p id="__p4">Multiresolution characteristics of coarse-grain noise. A noisy image is decomposed into its frequency subbands using <em>db</em>8 filters of Matlab. Part of the image is shown at the original resolution level and at three approximation subbands. The coarse-grain noise at the original level is difficult to identify and eliminate; the noise becomes fine grain as the image is decomposed, and can be eliminated more easily.</p></div></div></div><p id="P12" class="p p-last">The proposed framework is illustrated in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a>: A signal is decomposed into its frequency subbands with wavelet decomposition; as the signal is reconstructed back, bilateral filtering is applied to the approximation subbands. Unlike the standard single-level bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>], this multiresolution bilateral filter has the potential of eliminating low-frequency noise components. (This will become evident in our experiments with real data. Such an observation was also made in [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502068" role="button" aria-expanded="false" aria-haspopup="true">34</a>] for anisotropic diffusion, which has been shown to be related to the bilateral filter. Also, [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R31" rid="R31" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">31</a>] utilizes the bilateral filter (because it does not create halo artifacts) in a multiresolution scheme for shape and detail enhancement from multiple images.) Bilateral filtering works in approximation subbands; in addition, it is possible to apply wavelet thresholding to the detail subbands, where some noise components can be identified and removed effectively. This new image denoising framework combines bilateral filtering and wavelet thresholding. In the next section, we will demonstrate that this framework produces results better than the individual applications of the wavelet thresholding or the bilateral filter, or successive application of the wavelet thresholding and the bilateral filter. We will also discuss the contribution of the wavelet thresholding to overall performance.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139639968091792" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f5.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f5.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f5.jpg"></a></div><div id="largeobj_idm139639968091792" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Fig. 5</a></div><!--caption a7--><div class="caption"><p id="__p5">Illustration of the proposed method. An input image is decomposed into its approximation and detail subbands through wavelet decomposition. As the image is reconstructed back, bilateral filtering is applied to the approximation subbands, and wavelet thresholding is applied to the detail subbands. The analysis and synthesis filters (<em>L<sub>a,</sub> H<sub>a,</sub> L<sub>s,</sub></em> and <em>H<sub>s</sub></em>) form a perfect reconstruction filter bank. The illustration shows one approximation subband and one detail subband at each decomposition level; this would be the case when the data is one dimensional. For a two-dimensional data, there are in fact one approximation and three (horizontal, vertical, and diagonal) detail subbands at each decomposition level. Also, in the illustration, there are two levels of decomposition; the approximation subbands could be decomposed further in an application.</p></div></div></div></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">IV. Experiments and Discussions</h2><p id="P13" class="p p-first">We have conducted some experiments to see the performance of the proposed framework quantitatively and visually. To do a quantitative comparison, we simulated noisy images by adding white Gaussian noise with various standard deviations to some standard test images. These noisy images were then denoised using several algorithms and the PSNR results were calculated. For visual comparisons, real noisy images were used.</p><div id="S5" class="sec"><h3 id="S5title">A. PSNR Comparison for Simulated Noisy Images</h3><p id="P14" class="p p-first">For each test image, three noisy versions were created by adding white Gaussian noise with standard deviations 10, 20, and 30. PSNR results for six methods are included in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>. The first method is the BayesShrink wavelet thresholding algorithm [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502076" role="button" aria-expanded="false" aria-haspopup="true">3</a>]. Five decomposition levels were used; the noise variance was estimated using the robust median estimator [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R1" rid="R1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">1</a>]. The second method is the bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>]. Based on our experiments discussed in the previous sections, we chose the following parameters for the bilateral filter: <em>σ<sub>d</sub></em> = 1.8, <em>σ<sub>r</sub></em> = 2 × <em>σ<sub>n</sub></em>, and the window size is 11 × 11. The third method is the sequential application of [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502088" role="button" aria-expanded="false" aria-haspopup="true">3</a>] and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>]. The reason this method was included is to see the combined effect of [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502097" role="button" aria-expanded="false" aria-haspopup="true">3</a>] and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>] and compare it with the proposed method. The fourth method is the new SURE method of [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502103" role="button" aria-expanded="false" aria-haspopup="true">8</a>]. It was recently published, and shown to produce very good results with non-redundant wavelet decomposition. The fifth method is the 3D collaborative filtering (3DCF) of [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R13" rid="R13" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">13</a>], [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502086" role="button" aria-expanded="false" aria-haspopup="true">14</a>]. The sixth method is the proposed method. For the proposed method, <em>db8</em> filters in Matlab were used for one-level decomposition. For the bilateral filtering part of the proposed method, we set the parameters as follows: <em>σ<sub>d</sub></em> = 1.8, the window size is 11 × 11, and <em>σ<sub>r</sub></em> = 1.0 × <em>σ<sub>n</sub></em> at each level. (In case of the original bilateral filter, <em>σ<sub>r</sub></em> = 2 × <em>σ<sub>n</sub></em> was a better choice. However, for the proposed method this lead to a smaller PSNR value on average. The reason is the double application of the bilateral filter in the proposed method. When <em>σ<sub>r</sub></em> was large, the images were smoothed to produce low PSNR values. After some experimentation, <em>σ<sub>r</sub></em> = 1.0 × <em>σ<sub>n</sub></em> turned out to be a better choice in terms of PSNR values. Here, we should note that a higher PSNR does not necessarily correspond to a better visual quality. We will discuss this shortly.) For the wavelet thresholding part of the proposed method, the BayesShrink method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502101" role="button" aria-expanded="false" aria-haspopup="true">3</a>] was used; and the noise variance was estimated again with the robust median estimator technique. To eliminate the border effects, images were mirror-extended before the application of the bilateral filter and cropped to the original size at the end.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>TABLE I</h3><!--caption a7--><div class="caption"><p id="__p12">PSNR comparison of the BayesShrink method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502096" role="button" aria-expanded="false" aria-haspopup="true">3</a>], the bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>], sequential application of the BayesShrink [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502084" role="button" aria-expanded="false" aria-haspopup="true">3</a>] and the bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>] methods, New SURE thresholding [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502095" role="button" aria-expanded="false" aria-haspopup="true">8</a>], 3DCF [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502073" role="button" aria-expanded="false" aria-haspopup="true">14</a>] and the proposed method for simulated additive white Gaussian noise of various standard deviations. (The numbers were obtained by averaging the results of six runs.)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139640001823792" class="xtable"><table frame="box" rules="cols" class="rendered small default_table"><thead><tr><th align="center" rowspan="1" colspan="1">Input Image</th><th align="center" rowspan="1" colspan="1">σ<sub>n</sub></th><th align="center" rowspan="1" colspan="1">BayesShrink [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">3</a>]</th><th align="center" rowspan="1" colspan="1">Bilateral Filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>]</th><th align="center" rowspan="1" colspan="1">[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">3</a>] Followed by [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>]</th><th align="center" rowspan="1" colspan="1">New SURE [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R8" rid="R8" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">8</a>]</th><th align="center" rowspan="1" colspan="1">3DCF [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">14</a>]</th><th align="center" rowspan="1" colspan="1">Proposed Method</th></tr><tr><th colspan="8" align="center" valign="bottom" rowspan="1"><hr></th></tr></thead><tbody><tr><td rowspan="3" align="center" colspan="1">Barbara 512 × 512</td><td align="center" rowspan="1" colspan="1">10</td><td align="center" rowspan="1" colspan="1">31.25</td><td align="center" rowspan="1" colspan="1">31.37</td><td align="center" rowspan="1" colspan="1">30.92</td><td align="center" rowspan="1" colspan="1">32.18</td><td align="center" rowspan="1" colspan="1">34.98</td><td align="center" rowspan="1" colspan="1">31.79</td></tr><tr><td align="center" rowspan="1" colspan="1">20</td><td align="center" rowspan="1" colspan="1">27.32</td><td align="center" rowspan="1" colspan="1">27.02</td><td align="center" rowspan="1" colspan="1">27.16</td><td align="center" rowspan="1" colspan="1">27.98</td><td align="center" rowspan="1" colspan="1">31.78</td><td align="center" rowspan="1" colspan="1">27.74</td></tr><tr><td align="center" rowspan="1" colspan="1">30</td><td align="center" rowspan="1" colspan="1">25.34</td><td align="center" rowspan="1" colspan="1">24.69</td><td align="center" rowspan="1" colspan="1">25.23</td><td align="center" rowspan="1" colspan="1">25.83</td><td align="center" rowspan="1" colspan="1">29.81</td><td align="center" rowspan="1" colspan="1">25.61</td></tr><tr><td colspan="8" align="center" valign="bottom" rowspan="1">
<hr></td></tr><tr><td rowspan="3" align="center" colspan="1">Boats 512 × 512</td><td align="center" rowspan="1" colspan="1">10</td><td align="center" rowspan="1" colspan="1">31.98</td><td align="center" rowspan="1" colspan="1">32.02</td><td align="center" rowspan="1" colspan="1">31.81</td><td align="center" rowspan="1" colspan="1">32.90</td><td align="center" rowspan="1" colspan="1">33.92</td><td align="center" rowspan="1" colspan="1">32.58</td></tr><tr><td align="center" rowspan="1" colspan="1">20</td><td align="center" rowspan="1" colspan="1">28.55</td><td align="center" rowspan="1" colspan="1">28.40</td><td align="center" rowspan="1" colspan="1">28.43</td><td align="center" rowspan="1" colspan="1">29.47</td><td align="center" rowspan="1" colspan="1">30.88</td><td align="center" rowspan="1" colspan="1">29.25</td></tr><tr><td align="center" rowspan="1" colspan="1">30</td><td align="center" rowspan="1" colspan="1">26.71</td><td align="center" rowspan="1" colspan="1">26.57</td><td align="center" rowspan="1" colspan="1">26.66</td><td align="center" rowspan="1" colspan="1">27.63</td><td align="center" rowspan="1" colspan="1">29.12</td><td align="center" rowspan="1" colspan="1">27.24</td></tr><tr><td colspan="8" align="center" valign="bottom" rowspan="1">
<hr></td></tr><tr><td rowspan="3" align="center" colspan="1">Goldhill 512 × 512</td><td align="center" rowspan="1" colspan="1">10</td><td align="center" rowspan="1" colspan="1">31.94</td><td align="center" rowspan="1" colspan="1">32.08</td><td align="center" rowspan="1" colspan="1">31.93</td><td align="center" rowspan="1" colspan="1">32.69</td><td align="center" rowspan="1" colspan="1">33.62</td><td align="center" rowspan="1" colspan="1">32.48</td></tr><tr><td align="center" rowspan="1" colspan="1">20</td><td align="center" rowspan="1" colspan="1">28.69</td><td align="center" rowspan="1" colspan="1">28.90</td><td align="center" rowspan="1" colspan="1">28.80</td><td align="center" rowspan="1" colspan="1">29.52</td><td align="center" rowspan="1" colspan="1">30.72</td><td align="center" rowspan="1" colspan="1">29.50</td></tr><tr><td align="center" rowspan="1" colspan="1">30</td><td align="center" rowspan="1" colspan="1">27.13</td><td align="center" rowspan="1" colspan="1">27.50</td><td align="center" rowspan="1" colspan="1">27.34</td><td align="center" rowspan="1" colspan="1">27.89</td><td align="center" rowspan="1" colspan="1">29.16</td><td align="center" rowspan="1" colspan="1">27.77</td></tr><tr><td colspan="8" align="center" valign="bottom" rowspan="1">
<hr></td></tr><tr><td rowspan="3" align="center" colspan="1">Peppers 256 × 256</td><td align="center" rowspan="1" colspan="1">10</td><td align="center" rowspan="1" colspan="1">31.49</td><td align="center" rowspan="1" colspan="1">32.98</td><td align="center" rowspan="1" colspan="1">31.89</td><td align="center" rowspan="1" colspan="1">33.18</td><td align="center" rowspan="1" colspan="1">34.68</td><td align="center" rowspan="1" colspan="1">33.45</td></tr><tr><td align="center" rowspan="1" colspan="1">20</td><td align="center" rowspan="1" colspan="1">27.85</td><td align="center" rowspan="1" colspan="1">29.07</td><td align="center" rowspan="1" colspan="1">28.01</td><td align="center" rowspan="1" colspan="1">29.33</td><td align="center" rowspan="1" colspan="1">31.29</td><td align="center" rowspan="1" colspan="1">30.20</td></tr><tr><td align="center" rowspan="1" colspan="1">30</td><td align="center" rowspan="1" colspan="1">25.73</td><td align="center" rowspan="1" colspan="1">27.02</td><td align="center" rowspan="1" colspan="1">26.07</td><td align="center" rowspan="1" colspan="1">27.13</td><td align="center" rowspan="1" colspan="1">29.28</td><td align="center" rowspan="1" colspan="1">28.18</td></tr><tr><td colspan="8" align="center" valign="bottom" rowspan="1">
<hr></td></tr><tr><td rowspan="3" align="center" colspan="1">House 256 × 256</td><td align="center" rowspan="1" colspan="1">10</td><td align="center" rowspan="1" colspan="1">33.07</td><td align="center" rowspan="1" colspan="1">33.77</td><td align="center" rowspan="1" colspan="1">33.09</td><td align="center" rowspan="1" colspan="1">34.29</td><td align="center" rowspan="1" colspan="1">36.71</td><td align="center" rowspan="1" colspan="1">34.62</td></tr><tr><td align="center" rowspan="1" colspan="1">20</td><td align="center" rowspan="1" colspan="1">29.83</td><td align="center" rowspan="1" colspan="1">29.63</td><td align="center" rowspan="1" colspan="1">29.79</td><td align="center" rowspan="1" colspan="1">30.93</td><td align="center" rowspan="1" colspan="1">33.77</td><td align="center" rowspan="1" colspan="1">31.37</td></tr><tr><td align="center" rowspan="1" colspan="1">30</td><td align="center" rowspan="1" colspan="1">27.12</td><td align="center" rowspan="1" colspan="1">28.11</td><td align="center" rowspan="1" colspan="1">28.10</td><td align="center" rowspan="1" colspan="1">28.98</td><td align="center" rowspan="1" colspan="1">32.09</td><td align="center" rowspan="1" colspan="1">29.24</td></tr><tr><td colspan="8" align="center" valign="bottom" rowspan="1">
<hr></td></tr><tr><td rowspan="3" align="center" colspan="1">Lena 512 × 512</td><td align="center" rowspan="1" colspan="1">10</td><td align="center" rowspan="1" colspan="1">33.38</td><td align="center" rowspan="1" colspan="1">33.65</td><td align="center" rowspan="1" colspan="1">33.39</td><td align="center" rowspan="1" colspan="1">34.45</td><td align="center" rowspan="1" colspan="1">35.93</td><td align="center" rowspan="1" colspan="1">34.48</td></tr><tr><td align="center" rowspan="1" colspan="1">20</td><td align="center" rowspan="1" colspan="1">30.27</td><td align="center" rowspan="1" colspan="1">30.33</td><td align="center" rowspan="1" colspan="1">30.29</td><td align="center" rowspan="1" colspan="1">31.33</td><td align="center" rowspan="1" colspan="1">33.05</td><td align="center" rowspan="1" colspan="1">31.28</td></tr><tr><td align="center" rowspan="1" colspan="1">30</td><td align="center" rowspan="1" colspan="1">28.60</td><td align="center" rowspan="1" colspan="1">28.54</td><td align="center" rowspan="1" colspan="1">28.62</td><td align="center" rowspan="1" colspan="1">29.55</td><td align="center" rowspan="1" colspan="1">31.26</td><td align="center" rowspan="1" colspan="1">29.33</td></tr><tr><td colspan="8" align="center" valign="bottom" rowspan="1">
<hr></td></tr><tr><td align="center" rowspan="1" colspan="1"><strong>Average</strong></td><td align="center" rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">29.24</td><td align="center" rowspan="1" colspan="1">29.54</td><td align="center" rowspan="1" colspan="1">29.31</td><td align="center" rowspan="1" colspan="1">30.29</td><td align="center" rowspan="1" colspan="1">32.34</td><td align="center" rowspan="1" colspan="1">30.34</td></tr></tbody></table></div><div id="largeobj_idm139640001823792" class="largeobj-link align_right" style=""><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P15">As seen in the PSNR results of <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>, the proposed method is 0.8dB better than the original bilateral filter and 1.1dB better than the BayesShrink method on average. The sequential application of [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502058" role="button" aria-expanded="false" aria-haspopup="true">3</a>] and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>] is only slightly better than [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502080" role="button" aria-expanded="false" aria-haspopup="true">3</a>] and worse than [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>]. Therefore, we conclude that the improvement of the proposed method is not due to the combined effect of [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502069" role="button" aria-expanded="false" aria-haspopup="true">3</a>] and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>], but due to the multiresolution application of the bilateral filter. While the new SURE method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502063" role="button" aria-expanded="false" aria-haspopup="true">8</a>] is slightly worse than the proposed method, the 3DCF method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502057" role="button" aria-expanded="false" aria-haspopup="true">14</a>] is 2dB better than the proposed method.</p><p id="P16" class="p p-last">Most of the denoising methods are optimized for additive white Gaussian noise (AWGN); however, the real challenge is the performance for real noisy images. While we cannot quantitatively evaluate the performances for real noisy images, we may simulate spatially varying noise and make a quantitative comparison for it. This is more challenging than the AWGN case and could be a better representative of the performance for nonuniform noise situations. In our experiments, the space varying noise is generated by using a two-dimensional sinusoid of the same size as the input image; and the standard deviation of the noise at each pixel is controlled based on the amplitude of the sinusoid. To be specific, we generated the 2D signal <em>f</em>(<em>x</em><sub>1,</sub> <em>x</em><sub>2</sub>) = (<em>sin</em>(<em>x</em><sub>1</sub><em>/T</em>)<em>sin</em>(<em>x</em><sub>2</sub><em>/T</em>) + 1)/2, where <em>T</em> is the period of the sinusoid. For an input image <em>I</em>(<em>x</em><sub>1</sub>, <em>x</em><sub>2</sub>), the noisy image is <em>I</em>(<em>x</em><sub>1,</sub> <em>x</em><sub>2</sub>)+ <em>σ<sub>d</sub> f</em>(<em>x</em><sub>1,</sub> <em>x</em><sub>2</sub>). The experiment was repeated for several standard test images (for <em>T</em> = 10 and <em>σ<sub>d</sub></em> = 15); the results are shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table II</span></a>. Notice that methods specifically designed for additive white Gaussian noise do not perform well for this experiment. The neighborhood based denoising method of [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502078" role="button" aria-expanded="false" aria-haspopup="true">14</a>], which can be considered as an extension of the bilateral filter, is still the best; however, compared to the simulated AWGN noise experiments, the gap between the proposed method and [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502082" role="button" aria-expanded="false" aria-haspopup="true">14</a>] is much smaller. The standard bilateral filter also produces very good results. This experiment tells the effectiveness of the neighborhood based approach in case of space varying noise.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>TABLE II</h3><!--caption a7--><div class="caption"><p id="__p13">PSNR comparison of the BayesShrink method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502083" role="button" aria-expanded="false" aria-haspopup="true">3</a>], the bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>], 3DCF [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502091" role="button" aria-expanded="false" aria-haspopup="true">14</a>], New SURE thresholding [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502099" role="button" aria-expanded="false" aria-haspopup="true">8</a>], BLS-GSM [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502094" role="button" aria-expanded="false" aria-haspopup="true">4</a>], and the proposed method for the space varying noise simulation.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139639983526080" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th align="center" rowspan="1" colspan="1">Input Image</th><th align="center" rowspan="1" colspan="1">BayesShrink [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">3</a>]</th><th align="center" rowspan="1" colspan="1">Bilateral Filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>]</th><th align="center" rowspan="1" colspan="1">3DCF [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">14</a>]</th><th align="center" rowspan="1" colspan="1">New SURE [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R8" rid="R8" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">8</a>]</th><th align="center" rowspan="1" colspan="1">BLS-GSM [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R4" rid="R4" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">4</a>]</th><th align="center" rowspan="1" colspan="1">Proposed Method</th></tr></thead><tbody><tr><td align="center" rowspan="1" colspan="1">Barbara 512 × 512</td><td align="center" rowspan="1" colspan="1">31.9</td><td align="center" rowspan="1" colspan="1">32.2</td><td align="center" rowspan="1" colspan="1">34.3</td><td align="center" rowspan="1" colspan="1">24.0</td><td align="center" rowspan="1" colspan="1">29.7</td><td align="center" rowspan="1" colspan="1">32.6</td></tr><tr><td align="center" rowspan="1" colspan="1">Lena 512 × 512</td><td align="center" rowspan="1" colspan="1">33.1</td><td align="center" rowspan="1" colspan="1">33.4</td><td align="center" rowspan="1" colspan="1">35.2</td><td align="center" rowspan="1" colspan="1">22.7</td><td align="center" rowspan="1" colspan="1">31.2</td><td align="center" rowspan="1" colspan="1">34.1</td></tr><tr><td align="center" rowspan="1" colspan="1">Goldhill 512 × 512</td><td align="center" rowspan="1" colspan="1">32.2</td><td align="center" rowspan="1" colspan="1">32.6</td><td align="center" rowspan="1" colspan="1">32.3</td><td align="center" rowspan="1" colspan="1">23.6</td><td align="center" rowspan="1" colspan="1">29.9</td><td align="center" rowspan="1" colspan="1">32.8</td></tr><tr><td align="center" rowspan="1" colspan="1">Boat 512 × 512</td><td align="center" rowspan="1" colspan="1">32.5</td><td align="center" rowspan="1" colspan="1">32.6</td><td align="center" rowspan="1" colspan="1">32.9</td><td align="center" rowspan="1" colspan="1">23.6</td><td align="center" rowspan="1" colspan="1">30.3</td><td align="center" rowspan="1" colspan="1">32.9</td></tr><tr><td align="center" rowspan="1" colspan="1">House 256 × 256</td><td align="center" rowspan="1" colspan="1">32.5</td><td align="center" rowspan="1" colspan="1">33.4</td><td align="center" rowspan="1" colspan="1">35.8</td><td align="center" rowspan="1" colspan="1">22.7</td><td align="center" rowspan="1" colspan="1">31.4</td><td align="center" rowspan="1" colspan="1">34.1</td></tr><tr><td align="center" rowspan="1" colspan="1">Peppers 256 × 256</td><td align="center" rowspan="1" colspan="1">31.8</td><td align="center" rowspan="1" colspan="1">33.4</td><td align="center" rowspan="1" colspan="1">33.8</td><td align="center" rowspan="1" colspan="1">23.4</td><td align="center" rowspan="1" colspan="1">31.3</td><td align="center" rowspan="1" colspan="1">33.7</td></tr><tr><td align="center" rowspan="1" colspan="1">Average</td><td align="center" rowspan="1" colspan="1">32.3</td><td align="center" rowspan="1" colspan="1">32.9</td><td align="center" rowspan="1" colspan="1">34.0</td><td align="center" rowspan="1" colspan="1">23.3</td><td align="center" rowspan="1" colspan="1">30.6</td><td align="center" rowspan="1" colspan="1">33.4</td></tr></tbody></table></div><div id="largeobj_idm139639983526080" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="S6" class="sec sec-last"><h3 id="S6title">B. Visual Comparison for Real Noisy Images</h3><p id="P17" class="p p-first">PSNR comparisons with simulated white Gaussian noise tell only a part of the story: First, it is well known that the PSNR is not a very good measure of visual quality; second, the white Gaussian noise assumption is not always accurate for real images. As a result, experiments with real data and visual inspections are necessary to evaluate the real performance of image denoising algorithms.</p><p id="P18">In case of color images, there is also the issue of what color space to use. To achieve good PSNR performance, the RGB space could be a good choice; however, for visual performance, it is a better idea to perform denoising in the perceptually uniform <em>CIE-L</em><sup>*</sup><em>a</em><sup>*</sup><em>b</em><sup>*</sup> color space. As humans find color noise more objectionable than luminance noise, stronger noise filtering could be applied to the color channels <em>a</em><sup>*</sup> and <em>b</em><sup>*</sup> compared to the luminance channel <em>L</em><sup>*</sup> without making the image visually blurry. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figure 6</span></a> provides results of the proposed method for the RGB and the <em>L</em><sup>*</sup><em>a</em><sup>*</sup><em>b</em><sup>*</sup> spaces. For the RGB space result (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figure 6(b)</span></a>), the Euclidean distance between the (R,G,B) vectors is used as the intensity domain distance measure, and two levels of decomposition is applied to each channel. A close examination reveals that coarse-grain color artifacts are visible especially on the facial and hand regions. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figure 6(c)</span></a> is the result when the Euclidean distance between the (<em>L</em><sup>*</sup><em>,a</em><sup>*</sup><em>,b</em><sup>*</sup>) vectors is used, and again two levels of decomposition is applied to each channel. The results in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>figures 6(b) and 6(c)</span></a> are very similar. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figure 6(d)</span></a> is the result when each channel is treated separately, and when the number of decomposition levels is 1,3, and 3 for <em>L</em><sup>*</sup>, <em>a</em><sup>*</sup>, and <em>b</em><sup>*</sup> channels, respectively. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figure 6(e)</span></a> is the result when the Euclidean distance between the (<em>L</em><sup>*</sup><em>,a</em><sup>*</sup><em>,b</em><sup>*</sup>) vectors is used, and when the number of decomposition levels is 1,3, and 3 for <em>L</em><sup>*</sup>, <em>a</em><sup>*</sup>, and <em>b</em><sup>*</sup> channels, respectively. Notice that in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figures 6(d) and (e)</span></a>, texture is better preserved and coarse-grain noise is better eliminated compared to the previous results. Among the last two, there is not much observable difference. As a result, for perceptual reasons, we advocate applying and optimizing the denoising algorithms in the <em>L</em><sup>*</sup><em>a</em><sup>*</sup><em>b</em><sup>*</sup> space.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F6" co-legend-rid="lgnd_F6"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139639986936224" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f6.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f6.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f6.jpg"></a></div><div id="largeobj_idm139639986936224" class="largeobj-link align_right" style=""><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Fig. 6</a></div><!--caption a7--><div class="caption"><p id="__p6">Comparison of the proposed method for different intensity distance measures and the number of decomposition levels. (a) Input image. (b) The distance measure is the Euclidean distance between (R,G,B) vectors. The number of decomposition levels is 2 for all channels. (c) The distance measure is the Euclidean distance between (<em>L</em><sup>*</sup><em>a</em><sup>*</sup><em>b</em><sup>*</sup>) vectors. The number of decomposition levels is 2 for all channels. (d) Each channel is denoised separately in <em>L</em><sup>*</sup><em>a</em><sup>*</sup><em>b</em><sup>*</sup> space. The number of decomposition levels is 1 for the <em>L</em> channel and 3 for the <em>a</em><sup>*</sup> and <em>b</em><sup>*</sup> channels. (e) The distance measure is the Euclidean distance between (<em>L</em><sup>*</sup><em>a</em><sup>*</sup><em>b</em><sup>*</sup>) vectors. The number of decomposition levels is 1 for the <em>L</em> channel and 3 for the <em>a</em><sup>*</sup> and <em>b</em><sup>*</sup> channels. (In these experiments, <em>σ<sub>d</sub></em> = 3, <em>σ<sub>r</sub></em> = 3 × <em>σ<sub>n</sub></em>, and <em>σ<sub>n</sub></em> is estimated using the robust median estimator [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502061" role="button" aria-expanded="false" aria-haspopup="true">3</a>].</p></div></div></div><p id="P19">Next, we show a set of results for real noisy images. In all these experiments, <em>σ<sub>d</sub></em> and <em>σ<sub>r</sub></em> parameters fixed at <em>σ<sub>d</sub></em> = 1.8 and <em>σ<sub>r</sub></em> = 3 × <em>σ<sub>n</sub></em> produced very good results for the proposed method. (<em>σ<sub>n</sub></em> was estimated using the robust median estimator [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502056" role="button" aria-expanded="false" aria-haspopup="true">3</a>] for all images.) Therefore, we can claim that the proposed method is data-driven and robust for good visual performance.</p><p id="P20"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span>Figure 7</span></a> shows four test images. The first image is the blue channel of an image captured with Sony DCR-TRV27. The second image was captured with a Canon A530 at ISO 800. The other images were downloaded from [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R35" rid="R35" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">35</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F7" co-legend-rid="lgnd_F7"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139639986792944" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f7.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f7.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f7.jpg"></a></div><div id="largeobj_idm139639986792944" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F7/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Fig. 7</a></div><!--caption a7--><div class="caption"><p id="__p7">Input images to be denoised using various algorithms. The top-left image is the blue channel of an image captured with Sony DCR-TRV27. The bottom-left image was captured with a Canon A530 at ISO 800. The other images were downloaded from [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R35" rid="R35" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">35</a>].</p></div></div></div><p id="P21">In <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span>Figure 8</span></a>, we compare the standard bilateral filter, the 3DCF method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502093" role="button" aria-expanded="false" aria-haspopup="true">14</a>], and the proposed method. The input image was corrupted significantly with coarse-grain noise. The results show that the standard bilateral filter and the 3DCF method are not effective against the coarse-grain noise. We provide two results for the proposed method. <em>σ<sub>r</sub></em> = 2 × <em>σ<sub>n</sub></em> for one result; and <em>σ<sub>r</sub></em> = 3 × <em>σ<sub>n</sub></em> for the other. The coarse grain noise is reduced significantly in both cases. While more noise components are eliminated for larger <em>σ<sub>r</sub></em>, contouring artifacts may start to appear, which is a common problem of the bilateral filtering and anisotropic diffusion.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F8" co-legend-rid="lgnd_F8"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139640002451440" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f8.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f8.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f8.jpg"></a></div><div id="largeobj_idm139640002451440" class="largeobj-link align_right" style=""><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F8/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Fig. 8</a></div><!--caption a7--><div class="caption"><p id="__p8">From top to bottom: Result of (a) the bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>] with <em>σ<sub>d</sub></em> =1.8 and <em>σ<sub>r</sub></em> = 2 × <em>σ<sub>n</sub></em>, (b) the 3DCF method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502074" role="button" aria-expanded="false" aria-haspopup="true">14</a>], (c) the proposed method with the number of decomposition levels is 4, <em>σ<sub>d</sub></em> = 1.8, and <em>σ<sub>r</sub></em> = 2 × <em>σ<sub>n</sub></em> at each level, (d) the proposed method with the number of decomposition levels is 4, <em>σ<sub>d</sub></em> = 1.8, and <em>σ<sub>r</sub></em> = 3 × <em>σ<sub>n</sub></em> at each level. The subband decomposition filters are <em>db</em>8 in Matlab.</p></div></div></div><p id="P22">In <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Figure 9</span></a>, we compare the 3DCF method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502092" role="button" aria-expanded="false" aria-haspopup="true">14</a>], the standard bilateral filter and the proposed method. The standard bilateral filter was tested for various values of <em>σ<sub>d</sub></em> and <em>σ<sub>r</sub></em>. Some representative results are shown. As seen in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>figures 9(b–d)</span></a>, no matter what parameter values are chosen for the standard bilateral filter, the coarse-grain chroma noise could not be eliminated effectively. (We have also tested the iterative application of the bilateral filter. The results were not good either, and were not included in the figure.) Two results obtained by the proposed method are given: For the result in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Figure 9(e)</span></a>, the number of decomposition levels for the luminance channel is one; and in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Figure 9(f)</span></a> it is two. For both results, the number of decomposition levels for the chrominance channels is four. Coarse-grain chroma noise is eliminated in both cases. Increasing the number of decomposition levels for the luminance channel produces a smoother image as seen in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span>Figure 9(f)</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F9" co-legend-rid="lgnd_F9"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139640072938272" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f9.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f9.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f9.jpg"></a></div><div id="largeobj_idm139640072938272" class="largeobj-link align_right" style=""><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F9/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F9"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9">Fig. 9</a></div><!--caption a7--><div class="caption"><p id="__p9">(a) Result of the 3DCF method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502060" role="button" aria-expanded="false" aria-haspopup="true">14</a>], (b) The bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>] with <em>σ<sub>d</sub></em> = 1.8 and <em>σ<sub>r</sub></em> = 3 × <em>σ<sub>n</sub></em>, (c) The bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>] with <em>σ<sub>d</sub></em> = 1.8 and <em>σ<sub>r</sub></em> = 20 × <em>σ<sub>n</sub></em>, (d) The bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>] with <em>σ<sub>d</sub></em> = 5.0 and <em>σ<sub>r</sub></em> = 20 × <em>σ<sub>n</sub></em>, (e) The proposed method with the number of decomposition levels is (1, 4, 4) for the (<em>L</em><sub>*</sub>, <em>a</em><sub>*</sub>, <em>b</em><sub>*</sub>) channels, respectively. That is, the <em>L</em><sub>*</sub> channel is decomposed one level, and <em>a</em><sub>*</sub> and <em>b</em><sub>*</sub> channels are decomposed four levels. (f) The proposed method with the number of decomposition levels is (2, 4, 4) for the (<em>L</em><sub>*</sub>, <em>a</em><sub>*</sub>, <em>b</em><sub>*</sub>) channels, respectively. For the proposed method, <em>σ<sub>d</sub></em> = 1.8, and <em>σ<sub>r</sub></em> = 3 × <em>σ<sub>n</sub></em> at each level. The subband decomposition filters are <em>db</em>8 in Matlab. The noise parameter <em>σ<sub>n</sub></em> is estimated using the robust median estimator.</p></div></div></div><p id="P23">In <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figures 10</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and11,</span></span><span>11</span></a>, results of the 3DCF method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502067" role="button" aria-expanded="false" aria-haspopup="true">14</a>], the BLS-GSM method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502098" role="button" aria-expanded="false" aria-haspopup="true">4</a>], the bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>], and the proposed method are presented for real images provided at [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R35" rid="R35" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">35</a>]. Among these methods, the proposed method is apparently producing more visually pleasing results than the others. Notice the lack of color in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F10/" target="figure" class="fig-table-link figpopup" rid-figpopup="F10" rid-ob="ob-F10" co-legend-rid="lgnd_F10"><span>Figure 10</span></a> for the proposed method; this is due to the higher number of decomposition levels for the chrominance channels. If the number of decomposition levels is reduced, the result would be more colorful. In <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F11/" target="figure" class="fig-table-link figpopup" rid-figpopup="F11" rid-ob="ob-F11" co-legend-rid="lgnd_F11"><span>Figure 11</span></a>, noise was not completely eliminated by the 3DCF or the BLS-GSM methods. The result of the bilateral filter is less noisy but overly smoothed. The result of the proposed method can be considered as the best visual one among three.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F10" co-legend-rid="lgnd_F10"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139639997918928" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f10.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f10.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f10.jpg"></a></div><div id="largeobj_idm139639997918928" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F10/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F10"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F10/" target="figure" rid-figpopup="F10" rid-ob="ob-F10">Fig. 10</a></div><!--caption a7--><div class="caption"><p id="__p10">(a) Result of the 3DCF method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502089" role="button" aria-expanded="false" aria-haspopup="true">14</a>], (b) The BLS-GSM result obtained from [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R35" rid="R35" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">35</a>] (c) The bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>] result, (d) Result of the proposed method. For the bilateral filter, <em>σ<sub>d</sub></em> = 1.8, <em>σ<sub>r</sub></em> = 10 × <em>σ<sub>n</sub></em>, and the window size is 11 × 11. For the proposed method, <em>σ<sub>d</sub></em> = 1.8, <em>σ<sub>r</sub></em> = 3 × <em>σ<sub>n</sub></em> at each level, the window size is 11 × 11, and the number of decomposition levels is (1, 4, 4) for the (<em>L</em><sub>*</sub>, <em>a</em><sub>*</sub>, <em>b</em><sub>*</sub>) channels, respectively. The wavelet filters are <em>db</em>8 in Matlab. The noise parameter <em>σ<sub>n</sub></em> is estimated using the robust median estimator.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F11" co-legend-rid="lgnd_F11"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139640073659920" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2614560_nihms83523f11.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms83523f11.jpg" title="Click on image to zoom" class="tileshop" src="./Multiresolution Bilateral Filtering for Image Denoising_files/nihms83523f11.jpg"></a></div><div id="largeobj_idm139640073659920" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F11/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F11"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/figure/F11/" target="figure" rid-figpopup="F11" rid-ob="ob-F11">Fig. 11</a></div><!--caption a7--><div class="caption"><p id="__p11">(a) Result of the 3DCF method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502087" role="button" aria-expanded="false" aria-haspopup="true">14</a>], (b) The BLS-GSM result obtained from [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R35" rid="R35" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">35</a>] (c) The bilateral filter [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16" rid="R16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">16</a>] result, (d) Result of the proposed method. For the bilateral filter, <em>σ<sub>d</sub></em> = 1.8, <em>σ<sub>r</sub></em> = 10 × <em>σ<sub>n</sub></em>, and the window size is 11 × 11. For the proposed method, <em>σ<sub>d</sub></em> = 1.8, <em>σ<sub>r</sub></em> = 3 × <em>σ<sub>n</sub></em> at each level, the window size is 11 × 11, and the number of decomposition levels is (1, 4, 4) for the (<em>L</em><sub>*</sub>, <em>a</em><sub>*</sub>, <em>b</em><sub>*</sub>) channels, respectively. The wavelet filters are <em>db</em>8 in Matlab. The noise parameter <em>σ<sub>n</sub></em> is estimated using the robust median estimator.</p></div></div></div><p id="P24" class="p p-last">Finally, we should comment on the contribution of the wavelet thresholding to the multiresolution framework. (As mentioned earlier, in our experiments, we used the BayesShrink method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_318502062" role="button" aria-expanded="false" aria-haspopup="true">3</a>] for the wavelet thresholding part.) We have done experiments with and without the wavelet thresholding. For real image experiments, the difference is barely visible. That is, the dominant contribution is coming from the multiresolution bilateral filtering, and the contribution of wavelet thresholding is little. On the other hand, in additive white Gaussian noise simulations, wavelet thresholding has resulted in an improvement of about 0.5dB in PSNR. Considering all aspects, we did not want to exclude wavelet thresholding from the proposed framework because another wavelet thresholding method could produce better results; and we leave the investigation of this as a future work.</p></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">V. Conclusions</h2><p id="P25" class="p p-first">In this paper we make an empirical study of the optimal parameter values for the bilateral filter in image denoising applications and present a multiresolution image denoising framework, which integrates bilateral filtering and wavelet thresholding. In this framework, we decompose an image into low- and high-frequency components, and apply bilateral filtering on the approximation subbands and wavelet thresholding on the detail subbands. We have found that the optimal <em>σ<sub>r</sub></em> value of the bilateral filter is linearly proportional to the standard deviation of the noise. The optimal value of the <em>σ<sub>d</sub></em> is relatively independent of the noise power. Based on these results, we estimate the noise standard deviation at each level of the subband decomposition and use a constant multiple of it for the <em>σ<sub>r</sub></em> value of bilateral filtering. The experiments with real data demonstrate the effectiveness of the proposed method.</p><p id="P26">Note that in all real image experiments, <em>σ<sub>n</sub></em> values were estimated from the data, and the same <em>σ<sub>d</sub></em> and (<em>σ<sub>r</sub></em>/<em>σ<sub>n</sub></em>) values produced satisfactorily good results for the proposed method. That is, once the parameters were decided, there was no need to re-adjust them for another image.</p><p id="P27" class="p p-last">The key factor in the performance of the proposed method is the multiresolution application of the bilateral filter. It helped eliminating the coarse-grain noise in images. The wavelet thresholding adds power the proposed method as some noise components can be eliminated better in detail subbands. We used a specific wavelet thresholding technique (i.e., the BayesShrink method); it is possible to improve the results further by using better detail-subband-denoising techniques or using redundant wavelet decomposition. These issues and the detailed analysis of parameter selection for the proposed framework are left as future work. We believe that the proposed framework will inspire further research towards understanding and eliminating noise in real images and help better understanding of the bilateral filter.</p></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Acknowledgments</h2><div class="sec"><p id="__p14">This work was supported in part by the National Science Foundation under Grant No 0528785 and National Institutes of Health under Grant No 1R21AG032231-01.</p></div></div><div id="idm139639986413840" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139639986413840title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Donoho DL, Johnstone IM. Ideal spatial adaptation by wavelet shrinkage. <span><span class="ref-journal">Biometrika. </span><span class="ref-vol">81</span>(3):425–455. 1994.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrika&amp;title=Ideal+spatial+adaptation+by+wavelet+shrinkage&amp;author=DL+Donoho&amp;author=IM+Johnstone&amp;volume=81&amp;issue=3&amp;pages=425-455&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Donoho DL, Johnstone IM, Kerkyacharian G, Picard D. Wavelet shrinkage: Asymptopia? <span><span class="ref-journal">Journal of Royal Statistics Society, Series B. </span><span class="ref-vol">57</span>(2):301–369. 1995.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Royal+Statistics+Society,+Series+B&amp;title=Wavelet+shrinkage:+Asymptopia?&amp;author=DL+Donoho&amp;author=IM+Johnstone&amp;author=G+Kerkyacharian&amp;author=D+Picard&amp;volume=57&amp;issue=2&amp;pages=301-369&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Chang SG, Yu B, Vetterli M. Adaptive wavelet thresholding for image denoising and compression. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2000 September;<span class="ref-vol">9</span>(9):1532–1546.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=Adaptive+wavelet+thresholding+for+image+denoising+and+compression&amp;author=SG+Chang&amp;author=B+Yu&amp;author=M+Vetterli&amp;volume=9&amp;issue=9&amp;publication_year=2000&amp;pages=1532-1546&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Portilla J, Strela V, Wainwright MJ, Simoncelli EP. Image denoising using scale mixtures of gaussians in the wavelet domain. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2003 November;<span class="ref-vol">12</span>(11):1338–1351.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/18244692" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=Image+denoising+using+scale+mixtures+of+gaussians+in+the+wavelet+domain&amp;author=J+Portilla&amp;author=V+Strela&amp;author=MJ+Wainwright&amp;author=EP+Simoncelli&amp;volume=12&amp;issue=11&amp;publication_year=2003&amp;pages=1338-1351&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Pizurica A, Philips W. Estimating the probability of the presence of a signal of interest in multiresolution single- and multiband image denoising. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2006 March;<span class="ref-vol">15</span>(3):654–665.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16519352" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=Estimating+the+probability+of+the+presence+of+a+signal+of+interest+in+multiresolution+single-+and+multiband+image+denoising&amp;author=A+Pizurica&amp;author=W+Philips&amp;volume=15&amp;issue=3&amp;publication_year=2006&amp;pages=654-665&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Sendur L, Selesnick IW. Bivariate shrinkage functions for wavelet-based denoising exploiting interscale dependency. <span><span class="ref-journal">IEEE Trans Signal Processing. </span>2002 November;<span class="ref-vol">50</span>(11):2744–2756.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Signal+Processing&amp;title=Bivariate+shrinkage+functions+for+wavelet-based+denoising+exploiting+interscale+dependency&amp;author=L+Sendur&amp;author=IW+Selesnick&amp;volume=50&amp;issue=11&amp;publication_year=2002&amp;pages=2744-2756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Sendur L, Selesnick IW. Bivariate shrinkage with local variance estimation. <span><span class="ref-journal">IEEE Signal Processing Letters. </span>2002 December;<span class="ref-vol">9</span>(12):438–441.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Signal+Processing+Letters&amp;title=Bivariate+shrinkage+with+local+variance+estimation&amp;author=L+Sendur&amp;author=IW+Selesnick&amp;volume=9&amp;issue=12&amp;publication_year=2002&amp;pages=438-441&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Luisier F, Blu T, Unser M. A new sure approach to image denoising: Inter-scale orthonormal wavelet thresholding. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2007 March;<span class="ref-vol">16</span>(3):593–606.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17357721" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=A+new+sure+approach+to+image+denoising:+Inter-scale+orthonormal+wavelet+thresholding&amp;author=F+Luisier&amp;author=T+Blu&amp;author=M+Unser&amp;volume=16&amp;issue=3&amp;publication_year=2007&amp;pages=593-606&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Lyu S, Simoncelli EP.  Statistical modeling of images with fields of gaussian scale mixtures. In: Schölkopf B, Platt J, Hoffman T, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> Vol. 19. MIT Press; Cambridge, MA: 2007. pp. 945–952. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Advances+in+Neural+Information+Processing+Systems&amp;author=S+Lyu&amp;author=EP+Simoncelli&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Elad M, Aharon M. Image denoising via learned dictionaries and sparse representation. Proc. IEEE Computer Vision and Pattern Recognition; June 2006. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Elad+M+Aharon+M+Image+denoising+via+learned+dictionaries+and+sparse+representation+Proc.+IEEE+Computer+Vision+and+Pattern+Recognition+June+2006+" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Elad M, Aharon M. Image denoising via sparse and redundant representations over learned dictionaries. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2006 December;<span class="ref-vol">15</span>(12):3736–3745.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17153947" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=Image+denoising+via+sparse+and+redundant+representations+over+learned+dictionaries&amp;author=M+Elad&amp;author=M+Aharon&amp;volume=15&amp;issue=12&amp;publication_year=2006&amp;pages=3736-3745&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Mairal J, Elad M, Sapiro G. Sparse representation for color image restoration. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2008 January;<span class="ref-vol">17</span>(1):53–69.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/18229804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=Sparse+representation+for+color+image+restoration&amp;author=J+Mairal&amp;author=M+Elad&amp;author=G+Sapiro&amp;volume=17&amp;issue=1&amp;publication_year=2008&amp;pages=53-69&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Dabov K, Katkovnik V, Foi A, Egiazarian K. Image denoising with block-matching and 3d filtering. Proc. SPIE Electronic Imaging: Algorithms and Systems V; January 2006;  <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Dabov+K+Katkovnik+V+Foi+A+Egiazarian+K+Image+denoising+with+block-matching+and+3d+filtering+Proc.+SPIE+Electronic+Imaging:+Algorithms+and+Systems+V+January+2006+6064+" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Dabov K, Foi A, Katkovnik V, Egiazarian K. Image denoising by sparse 3d transform-domain collaborative filtering. <span><span class="ref-journal">IEEE Trans Image Processing. </span><span class="ref-vol">16</span>:2080–2095. 2007.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=Image+denoising+by+sparse+3d+transform-domain+collaborative+filtering&amp;author=K+Dabov&amp;author=A+Foi&amp;author=V+Katkovnik&amp;author=K+Egiazarian&amp;volume=16&amp;pages=2080-2095&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Hirakawa K, Parks TW. Image denoising using total least squares. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2006 September;<span class="ref-vol">15</span>(9):2730–2742.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16948317" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=Image+denoising+using+total+least+squares&amp;author=K+Hirakawa&amp;author=TW+Parks&amp;volume=15&amp;issue=9&amp;publication_year=2006&amp;pages=2730-2742&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Tomasi C, Manduchi R. Bilateral filtering for gray and color images. <span><span class="ref-journal">Proc Int Conf Computer Vision. </span>1998:839–846.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Int+Conf+Computer+Vision&amp;title=Bilateral+filtering+for+gray+and+color+images&amp;author=C+Tomasi&amp;author=R+Manduchi&amp;publication_year=1998&amp;pages=839-846&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Lee JS. Digital image smoothing and the sigma filter. <span><span class="ref-journal">CVGIP: Graphical Models and Image Processing. </span>1983 November;<span class="ref-vol">24</span>(2):255–269.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CVGIP:+Graphical+Models+and+Image+Processing&amp;title=Digital+image+smoothing+and+the+sigma+filter&amp;author=JS+Lee&amp;volume=24&amp;issue=2&amp;publication_year=1983&amp;pages=255-269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Yaroslavsky L.  <span class="ref-journal">Digital Picture Processing - An Introduction.</span> Springer Verlag; 1985.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Digital+Picture+Processing+-+An+Introduction&amp;author=L+Yaroslavsky&amp;publication_year=1985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Smith SM, Brady JM. Susan - a new approach to low level image processing. <span><span class="ref-journal">Int Journal of Computer Vision. </span>1997;<span class="ref-vol">23</span>:45–78.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+Journal+of+Computer+Vision&amp;title=Susan+-+a+new+approach+to+low+level+image+processing&amp;author=SM+Smith&amp;author=JM+Brady&amp;volume=23&amp;publication_year=1997&amp;pages=45-78&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Elad M. On the origin of the bilateral filter and ways to improve it. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2002 October;<span class="ref-vol">11</span>(10):1141–1151.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/18249686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=On+the+origin+of+the+bilateral+filter+and+ways+to+improve+it&amp;author=M+Elad&amp;volume=11&amp;issue=10&amp;publication_year=2002&amp;pages=1141-1151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Barash D. A fundamental relationship between bilateral filtering, adaptive smoothing, and the nonlinear diffusion equation. <span><span class="ref-journal">IEEE Trans Pattern Analysis and Machine Intelligence. </span>2002 June;<span class="ref-vol">24</span>(6):844–847.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Pattern+Analysis+and+Machine+Intelligence&amp;title=A+fundamental+relationship+between+bilateral+filtering,+adaptive+smoothing,+and+the+nonlinear+diffusion+equation&amp;author=D+Barash&amp;volume=24&amp;issue=6&amp;publication_year=2002&amp;pages=844-847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Buades A, Coll B, Morel J. Neighborhood filters and pde’s. <span><span class="ref-journal">Numerische Mathematik. </span><span class="ref-vol">105</span>:1–34. 2006.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Numerische+Mathematik&amp;title=Neighborhood+filters+and+pde%E2%80%99s&amp;author=A+Buades&amp;author=B+Coll&amp;author=J+Morel&amp;volume=105&amp;pages=1-34&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Sochen N, Kimmel R, Malladi R. A general framework for low level vision. <span><span class="ref-journal">IEEE Trans Image Processing. </span>1998 March;<span class="ref-vol">7</span>(3):310–318.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/18276251" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=A+general+framework+for+low+level+vision&amp;author=N+Sochen&amp;author=R+Kimmel&amp;author=R+Malladi&amp;volume=7&amp;issue=3&amp;publication_year=1998&amp;pages=310-318&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Sochen N, Kimmel R, Bruckstein AM. Diffusions and confusions in signal and image processing. <span><span class="ref-journal">Journal of Mathematical Imaging and Vision. </span><span class="ref-vol">14</span>(3):195–209. 2001.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Mathematical+Imaging+and+Vision&amp;title=Diffusions+and+confusions+in+signal+and+image+processing&amp;author=N+Sochen&amp;author=R+Kimmel&amp;author=AM+Bruckstein&amp;volume=14&amp;issue=3&amp;pages=195-209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Spira A, Kimmel R, Sochen N. A short time beltrami kernel for smoothing images and manifolds. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2007 June;<span class="ref-vol">16</span>(6):1628–1636.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17547140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=A+short+time+beltrami+kernel+for+smoothing+images+and+manifolds&amp;author=A+Spira&amp;author=R+Kimmel&amp;author=N+Sochen&amp;volume=16&amp;issue=6&amp;publication_year=2007&amp;pages=1628-1636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Kervrann C, Boulanger J. Optimal spatial adaptation for patch-based image denoising. <span><span class="ref-journal">IEEE Trans Image Processing. </span>2006 October;<span class="ref-vol">15</span>(10):2866–2878.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17022255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=Optimal+spatial+adaptation+for+patch-based+image+denoising&amp;author=C+Kervrann&amp;author=J+Boulanger&amp;volume=15&amp;issue=10&amp;publication_year=2006&amp;pages=2866-2878&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Durand F, Dorsey J. Fast bilateral filtering for the display of high-dynamic-range images. <span><span class="ref-journal">Proc SIGGRAPH. </span>2002:257–266.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+SIGGRAPH&amp;title=Fast+bilateral+filtering+for+the+display+of+high-dynamic-range+images&amp;author=F+Durand&amp;author=J+Dorsey&amp;publication_year=2002&amp;pages=257-266&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Eisemann E, Durand F. Flash photography enhancement via intrinsic relighting. <span><span class="ref-journal">Proc SIGGRAPH. </span>2004:673–678.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+SIGGRAPH&amp;title=Flash+photography+enhancement+via+intrinsic+relighting&amp;author=E+Eisemann&amp;author=F+Durand&amp;publication_year=2004&amp;pages=673-678&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Wong WCK, Chung ACS, Yu SCH. Trilateral filtering for biomedical images. <span><span class="ref-journal">Proc IEEE Int Symposium on Biomedical Imaging. </span>2004:820–823.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+IEEE+Int+Symposium+on+Biomedical+Imaging&amp;title=Trilateral+filtering+for+biomedical+images&amp;author=WCK+Wong&amp;author=ACS+Chung&amp;author=SCH+Yu&amp;publication_year=2004&amp;pages=820-823&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Bennett EP, McMillan L. Video enhancement using per-pixel virtual exposures. <span><span class="ref-journal">ACM Trans on Graphics. </span><span class="ref-vol">24</span>(3):845–852. 2005.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACM+Trans+on+Graphics&amp;title=Video+enhancement+using+per-pixel+virtual+exposures&amp;author=EP+Bennett&amp;author=L+McMillan&amp;volume=24&amp;issue=3&amp;pages=845-852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Fattal R, Agrawala M, Rusinkiewicz S. Multiscale shape and detail enhancement from multi-light image collections. <span><span class="ref-journal">ACM Transactions on Graphics (Proc SIGGRAPH) </span>2007 August;<span class="ref-vol">26</span>(3)</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACM+Transactions+on+Graphics+(Proc+SIGGRAPH)&amp;title=Multiscale+shape+and+detail+enhancement+from+multi-light+image+collections&amp;author=R+Fattal&amp;author=M+Agrawala&amp;author=S+Rusinkiewicz&amp;volume=26&amp;issue=3&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="citation">Elad M.  <span class="ref-journal">Scale-Space, Lecture-notes in Computer-Science.</span> Apr, 2005. Retinex by two bilateral filters; pp. 7–10. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Scale-Space,+Lecture-notes+in+Computer-Science&amp;author=M+Elad&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="citation">Paris S, Durand F. A fast approximation of the bilateral filter using a signal processing approach. <span><span class="ref-journal">Proc European Conference on Computer Vision. </span>2006:568–580.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+European+Conference+on+Computer+Vision&amp;title=A+fast+approximation+of+the+bilateral+filter+using+a+signal+processing+approach&amp;author=S+Paris&amp;author=F+Durand&amp;publication_year=2006&amp;pages=568-580&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="citation">Acton S. Multigrid anisotropic diffusion. <span><span class="ref-journal">IEEE Trans Image Processing. </span>1998 March;<span class="ref-vol">7</span>(3):280–291.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/18276248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Image+Processing&amp;title=Multigrid+anisotropic+diffusion&amp;author=S+Acton&amp;volume=7&amp;issue=3&amp;publication_year=1998&amp;pages=280-291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span>“Color test images,” [Available online:  <a href="http://decsai.ugr.es/javier/denoise" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://decsai.ugr.es/javier/denoise</a>], April 2008.</span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#idm139640070984272title">Abstract</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#S1title">I. Introduction</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#S2title">II. Parameter selection for the bilateral filter</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#S3title">III. A multiresolution image denoising framework</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#S4title">IV. Experiments and Discussions</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#S7title">V. Conclusions</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#S9title">Acknowledgments</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#idm139639986413840title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/epub/">ePub (beta)</a></li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/pdf/nihms83523.pdf">PDF (4.3M)</a></li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" data-citationid="PMC2614560" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2614560%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Multiresolution Bilateral Filtering for Image Denoising_files/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2614560%2F&amp;text=Multiresolution%20Bilateral%20Filtering%20for%20Image%20Denoising" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Multiresolution Bilateral Filtering for Image Denoising_files/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2614560%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Multiresolution Bilateral Filtering for Image Denoising_files/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19004705" data-aiid="2614560" data-aid="2614560" data-iid="175073" data-domainid="319" data-domain="nihpa" data-accid="PMC2614560" data-md5="a5d1dbd3ff5099bc03a81734719f964e">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" class="portlet_shutter ui-ncbitoggler shutter_closed" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-4" role="button" aria-expanded="false" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave" style="" id="ui-portlet_content-4" aria-hidden="true"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input type="hidden" name="absid" id="absid" value="2614560"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:&#39;#favUL&#39;,starExists:true,createCollectionUrl:&#39;#&#39;,manageCollectionsUrl:&#39;/myncbi/collections/&#39;"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="https://www.ncbi.nlm.nih.gov/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input type="submit" class="button_apply" name="addtocollection" value="1"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/20445823" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A New Image Denoising Framework Based on Bilateral Filter.</a><span class="source">[Proc SPIE Int Soc Opt Eng. 2008]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-6" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Zhang M, Gunturk BK. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Proc SPIE Int Soc Opt Eng. 2008 Jan 28; 6822:68221B. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/19586815" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Image denoising using trivariate shrinkage filter in the wavelet domain and joint bilateral filter in the spatial domain.</a><span class="source">[IEEE Trans Image Process. 2009]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Yu H, Zhao L, Wang H. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">IEEE Trans Image Process. 2009 Oct; 18(10):2364-9. Epub 2009 Jul 6.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/18390341" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">An optimized blockwise nonlocal means denoising filter for 3-D magnetic resonance images.</a><span class="source">[IEEE Trans Med Imaging. 2008]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Coupe P, Yger P, Prima S, Hellier P, Kervrann C, Barillot C. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">IEEE Trans Med Imaging. 2008 Apr; 27(4):425-41. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/30998899" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Filters: When, Why, and How (Not) to Use Them.</a><span class="source">[Neuron. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">de Cheveigné A, Nelken I. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Neuron. 2019 Apr 17; 102(2):280-293. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/30532667" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A Review of Denoising Medical Images Using Machine Learning  Approaches.</a><span class="source">[Curr Med Imaging Rev. 2018]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Kaur P, Singh G, Kaur P. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Curr Med Imaging Rev. 2018 Oct; 14(5):675-685. </em>
</div>
</li>
</ul>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=19004705&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=19004705&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-11" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-11" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6567112/" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Robust Noise Suppression Technique for a LADAR System via Eigenvalue-Based Adaptive Filtering</a><span class="source">[Sensors (Basel, Switzerland). ...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-12" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Xia X, Chen R, Wang P, Zhao Y. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Sensors (Basel, Switzerland). 2019 May 19; 19(10): 2311</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4555113/" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Dietary Assessment on a Mobile Phone Using Image Processing and Pattern Recognition Techniques: Algorithm Design and System Prototyping</a><span class="source">[Nutrients. 2015]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Probst Y, Nguyen DT, Tran MK, Li W. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nutrients. 2015 Jul 27; 7(8): 6128-6138</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4277560/" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Multiscale bilateral filtering for improving image quality in digital breast tomosynthesis</a><span class="source">[Medical Physics. 2014]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Lu Y, Chan HP, Wei J, Hadjiiski LM, Samala RK. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Medical Physics. 2014 Dec 22; 42(1): 182-195</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4138603/" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Measuring food intake with digital photography</a><span class="source">[Journal of human nutrition and...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Martin CK, Nicklas T, Gunturk B, Correa JB, Allen HR, Champagne C. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Journal of human nutrition and dietetics : the official journal of the British Dietetic Association. 2013 Jul 15; 27(0 1): 72-81</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3975169/" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Validity of the Remote Food Photography Method (RFPM) for estimating energy and nutrient intake in near real-time</a><span class="source">[Obesity (Silver Spring, Md.). ...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Martin CK, Correa JB, Han H, Allen HR, Rood J, Champagne CM, Gunturk BK, Bray GA. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Obesity (Silver Spring, Md.). 2011 Dec 1; 20(4): 891-899</em>
</div>
</li>
</ul>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/citedby/" ref="reftype=relart&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-17" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-17" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=2614560" ref="reftype=MedGen&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-18" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/19004705/" ref="reftype=PubMed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=2614560" ref="reftype=Taxonomy&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-21" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-21" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState(&#39;ClearHT&#39;)">Clear</a><a href="javascript:historyDisplayState(&#39;HTOff&#39;)" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState(&#39;HTOn&#39;)" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=5df5151739a66ee2bae59d88">Multiresolution Bilateral Filtering for Image Denoising</a><div class="ralinkpop offscreen_noflow">Multiresolution Bilateral Filtering for Image Denoising<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2008 Dec; 17(12)2324</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=5db8aa6875f66d6cfa023e33">Directional processing of color images: theory and experimental results.</a><div class="ralinkpop offscreen_noflow">Directional processing of color images: theory and experimental results.<div class="brieflinkpopdesc">IEEE Trans Image Process. 1996;5(6):868-80.</div>
</div>
<div class="tertiary">PubMed</div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState(&#39;HTOn&#39;)">Turn recording back on</a></p>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--P3" name="crb--P3" rid="P3" style="position: absolute; visibility: visible; width: 370px; top: 1717px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-22"><ul>
<li class="two_line" reference_id="R3">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Adaptive wavelet thresholding for image denoising and compression.</a><span class="one_line_source">[IEEE Trans Image Process.  2000]</span><div class="alt-note">
<div class="authors">Chang SG, Yu B, Vetterli M</div>
<div class="citation">IEEE Trans Image Process. 2000; 9(9):1532-46.</div>
</div>
</li>
<li class="two_line" reference_id="R4">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18244692/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising using scale mixtures of Gaussians in the wavelet domain.</a><span class="one_line_source">[IEEE Trans Image Process.  2003]</span><div class="alt-note">
<div class="authors">Portilla J, Strela V, Wainwright MJ, Simoncelli EP</div>
<div class="citation">IEEE Trans Image Process. 2003; 12(11):1338-51.</div>
</div>
</li>
<li class="two_line" reference_id="R5">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16519352/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Estimating the probability of the presence of a signal of interest in multiresolution single- and multiband image denoising.</a><span class="one_line_source">[IEEE Trans Image Process.  2006]</span><div class="alt-note">
<div class="authors">Pizurica A, Philips W</div>
<div class="citation">IEEE Trans Image Process. 2006 Mar; 15(3):654-65.</div>
</div>
</li>
<li class="two_line" reference_id="R8">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17357721/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A new SURE approach to image denoising: interscale orthonormal wavelet thresholding.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Luisier F, Blu T, Unser M</div>
<div class="citation">IEEE Trans Image Process. 2007 Mar; 16(3):593-606.</div>
</div>
</li>
<li class="two_line" reference_id="R11">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17153947/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising via sparse and redundant representations over learned dictionaries.</a><span class="one_line_source">[IEEE Trans Image Process.  2006]</span><div class="alt-note">
<div class="authors">Elad M, Aharon M</div>
<div class="citation">IEEE Trans Image Process. 2006 Dec; 15(12):3736-45.</div>
</div>
</li>
<li class="two_line" reference_id="R12">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18229804/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Sparse representation for color image restoration.</a><span class="one_line_source">[IEEE Trans Image Process.  2008]</span><div class="alt-note">
<div class="authors">Mairal J, Elad M, Sapiro G</div>
<div class="citation">IEEE Trans Image Process. 2008 Jan; 17(1):53-69.</div>
</div>
</li>
<li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li>
<li class="two_line" reference_id="R15">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16948317/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising using total least squares.</a><span class="one_line_source">[IEEE Trans Image Process.  2006]</span><div class="alt-note">
<div class="authors">Hirakawa K, Parks TW</div>
<div class="citation">IEEE Trans Image Process. 2006 Sep; 15(9):2730-42.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P5" name="crb--P5" rid="P5" style="position: absolute; visibility: visible; width: 370px; top: 2551px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-23"><ul>
<li class="two_line" reference_id="R20">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18249686/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">On the origin of the bilateral filter and ways to improve it.</a><span class="one_line_source">[IEEE Trans Image Process.  2002]</span><div class="alt-note">
<div class="authors">Elad M</div>
<div class="citation">IEEE Trans Image Process. 2002; 11(10):1141-51.</div>
</div>
</li>
<li class="two_line" reference_id="R23">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18276251/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A general framework for low level vision.</a><span class="one_line_source">[IEEE Trans Image Process.  1998]</span><div class="alt-note">
<div class="authors">Sochen N, Kimmel R, Malladi R</div>
<div class="citation">IEEE Trans Image Process. 1998; 7(3):310-8.</div>
</div>
</li>
<li class="two_line" reference_id="R25">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17547140/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A short- time beltrami kernel for smoothing images and manifolds.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Spira A, Kimmel R, Sochen N</div>
<div class="citation">IEEE Trans Image Process. 2007 Jun; 16(6):1628-36.</div>
</div>
</li>
<li class="two_line" reference_id="R26">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17022255/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Optimal spatial adaptation for patch-based image denoising.</a><span class="one_line_source">[IEEE Trans Image Process.  2006]</span><div class="alt-note">
<div class="authors">Kervrann C, Boulanger J</div>
<div class="citation">IEEE Trans Image Process. 2006 Oct; 15(10):2866-78.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P8" name="crb--P8" rid="P8" style="position: absolute; visibility: visible; width: 370px; top: 3190px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-24"><ul><li class="two_line" reference_id="R26">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17022255/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Optimal spatial adaptation for patch-based image denoising.</a><span class="one_line_source">[IEEE Trans Image Process.  2006]</span><div class="alt-note">
<div class="authors">Kervrann C, Boulanger J</div>
<div class="citation">IEEE Trans Image Process. 2006 Oct; 15(10):2866-78.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P12" name="crb--P12" rid="P12" style="position: absolute; visibility: visible; width: 370px; top: 6048px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-25"><ul><li class="two_line" reference_id="R34">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18276248/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multigrid anisotropic diffusion.</a><span class="one_line_source">[IEEE Trans Image Process.  1998]</span><div class="alt-note">
<div class="authors">Acton ST</div>
<div class="citation">IEEE Trans Image Process. 1998; 7(3):280-91.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P14" name="crb--P14" rid="P14" style="position: absolute; visibility: visible; width: 370px; top: 7070px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-26"><ul>
<li class="two_line" reference_id="R3">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Adaptive wavelet thresholding for image denoising and compression.</a><span class="one_line_source">[IEEE Trans Image Process.  2000]</span><div class="alt-note">
<div class="authors">Chang SG, Yu B, Vetterli M</div>
<div class="citation">IEEE Trans Image Process. 2000; 9(9):1532-46.</div>
</div>
</li>
<li class="two_line" reference_id="R8">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17357721/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A new SURE approach to image denoising: interscale orthonormal wavelet thresholding.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Luisier F, Blu T, Unser M</div>
<div class="citation">IEEE Trans Image Process. 2007 Mar; 16(3):593-606.</div>
</div>
</li>
<li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P15" name="crb--P15" rid="P15" style="position: absolute; visibility: visible; width: 370px; top: 8295px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-27"><ul>
<li class="two_line" reference_id="R3">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Adaptive wavelet thresholding for image denoising and compression.</a><span class="one_line_source">[IEEE Trans Image Process.  2000]</span><div class="alt-note">
<div class="authors">Chang SG, Yu B, Vetterli M</div>
<div class="citation">IEEE Trans Image Process. 2000; 9(9):1532-46.</div>
</div>
</li>
<li class="two_line" reference_id="R8">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17357721/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A new SURE approach to image denoising: interscale orthonormal wavelet thresholding.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Luisier F, Blu T, Unser M</div>
<div class="citation">IEEE Trans Image Process. 2007 Mar; 16(3):593-606.</div>
</div>
</li>
<li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P16" name="crb--P16" rid="P16" style="position: absolute; visibility: visible; width: 370px; top: 8435px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul><li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P19" name="crb--P19" rid="P19" style="position: absolute; visibility: visible; width: 370px; top: 10691px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul><li class="two_line" reference_id="R3">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Adaptive wavelet thresholding for image denoising and compression.</a><span class="one_line_source">[IEEE Trans Image Process.  2000]</span><div class="alt-note">
<div class="authors">Chang SG, Yu B, Vetterli M</div>
<div class="citation">IEEE Trans Image Process. 2000; 9(9):1532-46.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P21" name="crb--P21" rid="P21" style="position: absolute; visibility: visible; width: 370px; top: 11490px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul><li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P22" name="crb--P22" rid="P22" style="position: absolute; visibility: visible; width: 370px; top: 12423px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul><li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P23" name="crb--P23" rid="P23" style="position: absolute; visibility: visible; width: 370px; top: 13486px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-32"><ul>
<li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li>
<li class="two_line" reference_id="R4">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18244692/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising using scale mixtures of Gaussians in the wavelet domain.</a><span class="one_line_source">[IEEE Trans Image Process.  2003]</span><div class="alt-note">
<div class="authors">Portilla J, Strela V, Wainwright MJ, Simoncelli EP</div>
<div class="citation">IEEE Trans Image Process. 2003; 12(11):1338-51.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--P24" name="crb--P24" rid="P24" style="position: absolute; visibility: visible; width: 370px; top: 15055px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-33"><ul><li class="two_line" reference_id="R3">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Adaptive wavelet thresholding for image denoising and compression.</a><span class="one_line_source">[IEEE Trans Image Process.  2000]</span><div class="alt-note">
<div class="authors">Chang SG, Yu B, Vetterli M</div>
<div class="citation">IEEE Trans Image Process. 2000; 9(9):1532-46.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p6" name="crb--__p6" rid="__p6" style="position: absolute; visibility: visible; width: 370px; top: 10538px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-34"><ul><li class="two_line" reference_id="R3">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Adaptive wavelet thresholding for image denoising and compression.</a><span class="one_line_source">[IEEE Trans Image Process.  2000]</span><div class="alt-note">
<div class="authors">Chang SG, Yu B, Vetterli M</div>
<div class="citation">IEEE Trans Image Process. 2000; 9(9):1532-46.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p8" name="crb--__p8" rid="__p8" style="position: absolute; visibility: visible; width: 370px; top: 12378px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-35"><ul><li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p9" name="crb--__p9" rid="__p9" style="position: absolute; visibility: visible; width: 370px; top: 13397px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-36"><ul><li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p10" name="crb--__p10" rid="__p10" style="position: absolute; visibility: visible; width: 370px; top: 14332px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-37"><ul><li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p11" name="crb--__p11" rid="__p11" style="position: absolute; visibility: visible; width: 370px; top: 15010px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-38"><ul><li class="two_line" reference_id="R14">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p12" name="crb--__p12" rid="__p12" style="position: absolute; visibility: visible; width: 370px; top: 7646px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-39"><ul>
<li class="two_line" reference_id="R3">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Adaptive wavelet thresholding for image denoising and compression.</a><span class="one_line_source">[IEEE Trans Image Process.  2000]</span><div class="alt-note">
<div class="authors">Chang SG, Yu B, Vetterli M</div>
<div class="citation">IEEE Trans Image Process. 2000; 9(9):1532-46.</div>
</div>
</li>
<li class="two_line" reference_id="R8" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17357721/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A new SURE approach to image denoising: interscale orthonormal wavelet thresholding.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Luisier F, Blu T, Unser M</div>
<div class="citation">IEEE Trans Image Process. 2007 Mar; 16(3):593-606.</div>
</div>
</li>
<li class="two_line" reference_id="R14" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li>
</ul><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/" class="seemore">See more ...</a></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p13" name="crb--__p13" rid="__p13" style="position: absolute; visibility: visible; width: 370px; top: 8912px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-40"><ul>
<li class="two_line" reference_id="R3" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Adaptive wavelet thresholding for image denoising and compression.</a><span class="one_line_source">[IEEE Trans Image Process.  2000]</span><div class="alt-note">
<div class="authors">Chang SG, Yu B, Vetterli M</div>
<div class="citation">IEEE Trans Image Process. 2000; 9(9):1532-46.</div>
</div>
</li>
<li class="two_line" reference_id="R14" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising by sparse 3-D transform-domain collaborative filtering.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div>
<div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div>
</div>
</li>
<li class="two_line" reference_id="R8" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17357721/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A new SURE approach to image denoising: interscale orthonormal wavelet thresholding.</a><span class="one_line_source">[IEEE Trans Image Process.  2007]</span><div class="alt-note">
<div class="authors">Luisier F, Blu T, Unser M</div>
<div class="citation">IEEE Trans Image Process. 2007 Mar; 16(3):593-606.</div>
</div>
</li>
<li class="two_line" reference_id="R4" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18244692/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image denoising using scale mixtures of Gaussians in the wavelet domain.</a><span class="one_line_source">[IEEE Trans Image Process.  2003]</span><div class="alt-note">
<div class="authors">Portilla J, Strela V, Wainwright MJ, Simoncelli EP</div>
<div class="citation">IEEE Trans Image Process. 2003; 12(11):1338-51.</div>
</div>
</li>
</ul><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/" class="seemore">See links ...</a></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="./Multiresolution Bilateral Filtering for Image Denoising_files/jquery.scrollTo-1.4.2.js.download"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-R1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">1. <span class="citation">Donoho DL, Johnstone IM. Ideal spatial adaptation by wavelet shrinkage. <span><span class="ref-journal">Biometrika. </span><span class="ref-vol">81</span>(3):425–455. 1994.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrika&amp;title=Ideal+spatial+adaptation+by+wavelet+shrinkage&amp;author=DL+Donoho&amp;author=IM+Johnstone&amp;volume=81&amp;issue=3&amp;pages=425-455&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R1">Ref list</a>]</div><div id="body-link-popper-R2" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">2. <span class="citation">Donoho DL, Johnstone IM, Kerkyacharian G, Picard D. Wavelet shrinkage: Asymptopia? <span><span class="ref-journal">Journal of Royal Statistics Society, Series B. </span><span class="ref-vol">57</span>(2):301–369. 1995.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Royal+Statistics+Society,+Series+B&amp;title=Wavelet+shrinkage:+Asymptopia?&amp;author=DL+Donoho&amp;author=IM+Johnstone&amp;author=G+Kerkyacharian&amp;author=D+Picard&amp;volume=57&amp;issue=2&amp;pages=301-369&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R2">Ref list</a>]</div><div id="body-link-popper-R3" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Adaptive wavelet thresholding for image denoising and compression.<div class="authors">Chang SG, Yu B, Vetterli M</div><div class="citation">IEEE Trans Image Process. 2000; 9(9):1532-46.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18262991/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R3">Ref list</a>]</p></div><div id="body-link-popper-R4" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Image denoising using scale mixtures of Gaussians in the wavelet domain.<div class="authors">Portilla J, Strela V, Wainwright MJ, Simoncelli EP</div><div class="citation">IEEE Trans Image Process. 2003; 12(11):1338-51.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18244692/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R4">Ref list</a>]</p></div><div id="body-link-popper-R5" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Estimating the probability of the presence of a signal of interest in multiresolution single- and multiband image denoising.<div class="authors">Pizurica A, Philips W</div><div class="citation">IEEE Trans Image Process. 2006 Mar; 15(3):654-65.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16519352/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R5">Ref list</a>]</p></div><div id="body-link-popper-R6" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">6. <span class="citation">Sendur L, Selesnick IW. Bivariate shrinkage functions for wavelet-based denoising exploiting interscale dependency. <span><span class="ref-journal">IEEE Trans Signal Processing. </span>2002 November;<span class="ref-vol">50</span>(11):2744–2756.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Signal+Processing&amp;title=Bivariate+shrinkage+functions+for+wavelet-based+denoising+exploiting+interscale+dependency&amp;author=L+Sendur&amp;author=IW+Selesnick&amp;volume=50&amp;issue=11&amp;publication_year=2002&amp;pages=2744-2756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R6">Ref list</a>]</div><div id="body-link-popper-R7" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">7. <span class="citation">Sendur L, Selesnick IW. Bivariate shrinkage with local variance estimation. <span><span class="ref-journal">IEEE Signal Processing Letters. </span>2002 December;<span class="ref-vol">9</span>(12):438–441.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Signal+Processing+Letters&amp;title=Bivariate+shrinkage+with+local+variance+estimation&amp;author=L+Sendur&amp;author=IW+Selesnick&amp;volume=9&amp;issue=12&amp;publication_year=2002&amp;pages=438-441&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R7">Ref list</a>]</div><div id="body-link-popper-R8" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A new SURE approach to image denoising: interscale orthonormal wavelet thresholding.<div class="authors">Luisier F, Blu T, Unser M</div><div class="citation">IEEE Trans Image Process. 2007 Mar; 16(3):593-606.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17357721/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R8">Ref list</a>]</p></div><div id="body-link-popper-R9" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">9. <span class="citation">Lyu S, Simoncelli EP.  Statistical modeling of images with fields of gaussian scale mixtures. In: Schölkopf B, Platt J, Hoffman T, editors. <span class="ref-journal">Advances in Neural Information Processing Systems.</span> Vol. 19. MIT Press; Cambridge, MA: 2007. pp. 945–952. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Advances+in+Neural+Information+Processing+Systems&amp;author=S+Lyu&amp;author=EP+Simoncelli&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R9">Ref list</a>]</div><div id="body-link-popper-R10" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">10. <span class="citation">Elad M, Aharon M. Image denoising via learned dictionaries and sparse representation. Proc. IEEE Computer Vision and Pattern Recognition; June 2006. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Elad+M+Aharon+M+Image+denoising+via+learned+dictionaries+and+sparse+representation+Proc.+IEEE+Computer+Vision+and+Pattern+Recognition+June+2006+" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R10">Ref list</a>]</div><div id="body-link-popper-R11" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Image denoising via sparse and redundant representations over learned dictionaries.<div class="authors">Elad M, Aharon M</div><div class="citation">IEEE Trans Image Process. 2006 Dec; 15(12):3736-45.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17153947/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R11">Ref list</a>]</p></div><div id="body-link-popper-R12" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Sparse representation for color image restoration.<div class="authors">Mairal J, Elad M, Sapiro G</div><div class="citation">IEEE Trans Image Process. 2008 Jan; 17(1):53-69.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18229804/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R12">Ref list</a>]</p></div><div id="body-link-popper-R13" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">13. <span class="citation">Dabov K, Katkovnik V, Foi A, Egiazarian K. Image denoising with block-matching and 3d filtering. Proc. SPIE Electronic Imaging: Algorithms and Systems V; January 2006;  <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Dabov+K+Katkovnik+V+Foi+A+Egiazarian+K+Image+denoising+with+block-matching+and+3d+filtering+Proc.+SPIE+Electronic+Imaging:+Algorithms+and+Systems+V+January+2006+6064+" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R13">Ref list</a>]</div><div id="body-link-popper-R14" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Image denoising by sparse 3-D transform-domain collaborative filtering.<div class="authors">Dabov K, Foi A, Katkovnik V, Egiazarian K</div><div class="citation">IEEE Trans Image Process. 2007 Aug; 16(8):2080-95.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17688213/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R14">Ref list</a>]</p></div><div id="body-link-popper-R15" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Image denoising using total least squares.<div class="authors">Hirakawa K, Parks TW</div><div class="citation">IEEE Trans Image Process. 2006 Sep; 15(9):2730-42.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16948317/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R15">Ref list</a>]</p></div><div id="body-link-popper-R16" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">16. <span class="citation">Tomasi C, Manduchi R. Bilateral filtering for gray and color images. <span><span class="ref-journal">Proc Int Conf Computer Vision. </span>1998:839–846.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Int+Conf+Computer+Vision&amp;title=Bilateral+filtering+for+gray+and+color+images&amp;author=C+Tomasi&amp;author=R+Manduchi&amp;publication_year=1998&amp;pages=839-846&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R16">Ref list</a>]</div><div id="body-link-popper-R17" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">17. <span class="citation">Lee JS. Digital image smoothing and the sigma filter. <span><span class="ref-journal">CVGIP: Graphical Models and Image Processing. </span>1983 November;<span class="ref-vol">24</span>(2):255–269.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CVGIP:+Graphical+Models+and+Image+Processing&amp;title=Digital+image+smoothing+and+the+sigma+filter&amp;author=JS+Lee&amp;volume=24&amp;issue=2&amp;publication_year=1983&amp;pages=255-269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R17">Ref list</a>]</div><div id="body-link-popper-R18" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">18. <span class="citation">Yaroslavsky L.  <span class="ref-journal">Digital Picture Processing - An Introduction.</span> Springer Verlag; 1985.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Digital+Picture+Processing+-+An+Introduction&amp;author=L+Yaroslavsky&amp;publication_year=1985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R18">Ref list</a>]</div><div id="body-link-popper-R19" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">19. <span class="citation">Smith SM, Brady JM. Susan - a new approach to low level image processing. <span><span class="ref-journal">Int Journal of Computer Vision. </span>1997;<span class="ref-vol">23</span>:45–78.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+Journal+of+Computer+Vision&amp;title=Susan+-+a+new+approach+to+low+level+image+processing&amp;author=SM+Smith&amp;author=JM+Brady&amp;volume=23&amp;publication_year=1997&amp;pages=45-78&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R19">Ref list</a>]</div><div id="body-link-popper-R20" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">On the origin of the bilateral filter and ways to improve it.<div class="authors">Elad M</div><div class="citation">IEEE Trans Image Process. 2002; 11(10):1141-51.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18249686/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R20">Ref list</a>]</p></div><div id="body-link-popper-R21" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">21. <span class="citation">Barash D. A fundamental relationship between bilateral filtering, adaptive smoothing, and the nonlinear diffusion equation. <span><span class="ref-journal">IEEE Trans Pattern Analysis and Machine Intelligence. </span>2002 June;<span class="ref-vol">24</span>(6):844–847.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Pattern+Analysis+and+Machine+Intelligence&amp;title=A+fundamental+relationship+between+bilateral+filtering,+adaptive+smoothing,+and+the+nonlinear+diffusion+equation&amp;author=D+Barash&amp;volume=24&amp;issue=6&amp;publication_year=2002&amp;pages=844-847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R21">Ref list</a>]</div><div id="body-link-popper-R22" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">22. <span class="citation">Buades A, Coll B, Morel J. Neighborhood filters and pde’s. <span><span class="ref-journal">Numerische Mathematik. </span><span class="ref-vol">105</span>:1–34. 2006.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Numerische+Mathematik&amp;title=Neighborhood+filters+and+pde%E2%80%99s&amp;author=A+Buades&amp;author=B+Coll&amp;author=J+Morel&amp;volume=105&amp;pages=1-34&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R22">Ref list</a>]</div><div id="body-link-popper-R23" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A general framework for low level vision.<div class="authors">Sochen N, Kimmel R, Malladi R</div><div class="citation">IEEE Trans Image Process. 1998; 7(3):310-8.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18276251/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R23">Ref list</a>]</p></div><div id="body-link-popper-R24" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">24. <span class="citation">Sochen N, Kimmel R, Bruckstein AM. Diffusions and confusions in signal and image processing. <span><span class="ref-journal">Journal of Mathematical Imaging and Vision. </span><span class="ref-vol">14</span>(3):195–209. 2001.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Mathematical+Imaging+and+Vision&amp;title=Diffusions+and+confusions+in+signal+and+image+processing&amp;author=N+Sochen&amp;author=R+Kimmel&amp;author=AM+Bruckstein&amp;volume=14&amp;issue=3&amp;pages=195-209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R24">Ref list</a>]</div><div id="body-link-popper-R25" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A short- time beltrami kernel for smoothing images and manifolds.<div class="authors">Spira A, Kimmel R, Sochen N</div><div class="citation">IEEE Trans Image Process. 2007 Jun; 16(6):1628-36.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17547140/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R25">Ref list</a>]</p></div><div id="body-link-popper-R26" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Optimal spatial adaptation for patch-based image denoising.<div class="authors">Kervrann C, Boulanger J</div><div class="citation">IEEE Trans Image Process. 2006 Oct; 15(10):2866-78.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17022255/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R26">Ref list</a>]</p></div><div id="body-link-popper-R27" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">27. <span class="citation">Durand F, Dorsey J. Fast bilateral filtering for the display of high-dynamic-range images. <span><span class="ref-journal">Proc SIGGRAPH. </span>2002:257–266.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+SIGGRAPH&amp;title=Fast+bilateral+filtering+for+the+display+of+high-dynamic-range+images&amp;author=F+Durand&amp;author=J+Dorsey&amp;publication_year=2002&amp;pages=257-266&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R27">Ref list</a>]</div><div id="body-link-popper-R28" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">28. <span class="citation">Eisemann E, Durand F. Flash photography enhancement via intrinsic relighting. <span><span class="ref-journal">Proc SIGGRAPH. </span>2004:673–678.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+SIGGRAPH&amp;title=Flash+photography+enhancement+via+intrinsic+relighting&amp;author=E+Eisemann&amp;author=F+Durand&amp;publication_year=2004&amp;pages=673-678&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R28">Ref list</a>]</div><div id="body-link-popper-R29" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">29. <span class="citation">Wong WCK, Chung ACS, Yu SCH. Trilateral filtering for biomedical images. <span><span class="ref-journal">Proc IEEE Int Symposium on Biomedical Imaging. </span>2004:820–823.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+IEEE+Int+Symposium+on+Biomedical+Imaging&amp;title=Trilateral+filtering+for+biomedical+images&amp;author=WCK+Wong&amp;author=ACS+Chung&amp;author=SCH+Yu&amp;publication_year=2004&amp;pages=820-823&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R29">Ref list</a>]</div><div id="body-link-popper-R30" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">30. <span class="citation">Bennett EP, McMillan L. Video enhancement using per-pixel virtual exposures. <span><span class="ref-journal">ACM Trans on Graphics. </span><span class="ref-vol">24</span>(3):845–852. 2005.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACM+Trans+on+Graphics&amp;title=Video+enhancement+using+per-pixel+virtual+exposures&amp;author=EP+Bennett&amp;author=L+McMillan&amp;volume=24&amp;issue=3&amp;pages=845-852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R30">Ref list</a>]</div><div id="body-link-popper-R31" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">31. <span class="citation">Fattal R, Agrawala M, Rusinkiewicz S. Multiscale shape and detail enhancement from multi-light image collections. <span><span class="ref-journal">ACM Transactions on Graphics (Proc SIGGRAPH) </span>2007 August;<span class="ref-vol">26</span>(3)</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=ACM+Transactions+on+Graphics+(Proc+SIGGRAPH)&amp;title=Multiscale+shape+and+detail+enhancement+from+multi-light+image+collections&amp;author=R+Fattal&amp;author=M+Agrawala&amp;author=S+Rusinkiewicz&amp;volume=26&amp;issue=3&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R31">Ref list</a>]</div><div id="body-link-popper-R32" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">32. <span class="citation">Elad M.  <span class="ref-journal">Scale-Space, Lecture-notes in Computer-Science.</span> Apr, 2005. Retinex by two bilateral filters; pp. 7–10. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Scale-Space,+Lecture-notes+in+Computer-Science&amp;author=M+Elad&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R32">Ref list</a>]</div><div id="body-link-popper-R33" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">33. <span class="citation">Paris S, Durand F. A fast approximation of the bilateral filter using a signal processing approach. <span><span class="ref-journal">Proc European Conference on Computer Vision. </span>2006:568–580.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+European+Conference+on+Computer+Vision&amp;title=A+fast+approximation+of+the+bilateral+filter+using+a+signal+processing+approach&amp;author=S+Paris&amp;author=F+Durand&amp;publication_year=2006&amp;pages=568-580&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R33">Ref list</a>]</div><div id="body-link-popper-R34" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Multigrid anisotropic diffusion.<div class="authors">Acton ST</div><div class="citation">IEEE Trans Image Process. 1998; 7(3):280-91.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18276248/" ref="reftype=pubmed&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R34">Ref list</a>]</p></div><div id="body-link-popper-R35" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">35. <span>“Color test images,” [Available online:  <a href="http://decsai.ugr.es/javier/denoise" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=2614560&amp;issue-id=175073&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://decsai.ugr.es/javier/denoise</a>], April 2008.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#R35">Ref list</a>]</div></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-12-14T12:00:07-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE898CB2DF507F310000000005EA036D&amp;ncbi_session=CE8B70D6DB891011_1286SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2614560%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="./Multiresolution Bilateral Filtering for Image Denoising_files/preloaderWidget.js.download"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov/" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov/" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov/" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov/" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov/">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="https://www.ncbi.nlm.nih.gov/home/about/policies.shtml">Policies and Guidelines</a> | <a href="https://www.ncbi.nlm.nih.gov/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="./Multiresolution Bilateral Filtering for Image Denoising_files/InstrumentPageStarterJS.js.download"> </script>    
    <script type="text/javascript" src="./Multiresolution Bilateral Filtering for Image Denoising_files/hfjs2.js.download"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 64.6px; left: 458.8px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614560/#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2614560/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B70D6DB891011_1286SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="./Multiresolution Bilateral Filtering for Image Denoising_files/4065628.js.download" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1" style="display: none;"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-family: MathJax_Size1, sans-serif;"></div></div></body></html>